
# Awesome RNA Virus Tools  
<img src="rdrp_summit.png" alt="drawing" width="200"/>  

> [!WARNING]  
THis repo is new and has bugs and errors, feel free to help out  

[![Awesome](https://awesome.re/badge.svg)](https://awesome.re) [![Website](https://img.shields.io/website?url=https%3A//rdrp-summit.github.io/awesome-rna-virus-tools/)](https://rdrp-summit.github.io/awesome-rna-virus-tools/) [![GitHub](https://img.shields.io/github/license/rdrp-summit/awesome-rna-virus-tools)](LICENSE)
  
A curated list of software, tools, databases and resources for RNA virus analysis, prediction, annotation, phylogenetics, and related research. This project builds upon the excellent [awesome-virome](https://github.com/shandley/awesome-virome) list.

üåê **[Visit the interactive website](https://rdrp-summit.github.io/awesome-rna-virus-tools/)** to explore tools by categories, programming languages, etc.
    
- [Tools](#tool)
  - [Multi-Classifier](#tool-multi-classifier)
  - [Phylogenetics](#tool-phylogenetics)
  - [Deep Learning](#tool-deep-learning)
  - [Metagenomic Input](#tool-metagenomic-input)
  - [Genome-Annotation](#tool-genome-annotation)
  - [Assembly](#tool-assembly)
  - [Serratus](#tool-serratus)
  - [Characterization](#tool-characterization)
  - [Annotation](#tool-annotation)
  - [Genome Reconstruction](#tool-genome-reconstruction)

- [Databases](#database)
  - [Database](#database-database)
  - [Serratus](#database-serratus)

- [Papers](#paper)
  - [Fish](#paper-fish)
  - [Arthropods](#paper-arthropods)
  - [Taxonomy](#paper-taxonomy)
  - [Marine](#paper-marine)
  - [Phages](#paper-phages)
  - [Aves](#paper-aves)
  - [Evolution](#paper-evolution)
  - [Phylogeny](#paper-phylogeny)
  - [Mammalia](#paper-mammalia)
  - [Gastropods](#paper-gastropods)
  - [Insects](#paper-insects)
  - [Parasite](#paper-parasite)
  - [Mosquito](#paper-mosquito)
  - [Hot-Springs](#paper-hot-springs)
  - [Soil](#paper-soil)
  - [Flavi](#paper-flavi)
  - [Amphibians](#paper-amphibians)
  - [Mosquitoes](#paper-mosquitoes)
  - [Surveillance](#paper-surveillance)
  - [-Ssrna](#paper--ssrna)
  - [Hexapods](#paper-hexapods)
  - [Ecology](#paper-ecology)
  - [Plants](#paper-plants)
  - [Viroids](#paper-viroids)
  - [Animalia](#paper-animalia)
  - [Invertebrate](#paper-invertebrate)
  - [Insect](#paper-insect)
  - [Human](#paper-human)
  - [Bats](#paper-bats)
  - [Fungi](#paper-fungi)
  - [Mollosca](#paper-mollosca)
  - [Hepacivirus](#paper-hepacivirus)
  - [Serratus](#paper-serratus)
  - [Diptera](#paper-diptera)
  - [Orthomyxoviridae](#paper-orthomyxoviridae)
  - [Resource](#paper-resource)
  - [Review](#paper-review)
  - [Reptiles](#paper-reptiles)
  - [Methodology](#paper-methodology)
  - [Algae](#paper-algae)
  - [Vertebrate](#paper-vertebrate)
  - [Flaviviridae](#paper-flaviviridae)

## Tools <a id='tool'></a>
### Multi-Classifier <a id='tool-multi-classifier'></a>
- **[VirSorter2: a multi-classifier, expert-guided approach to detect diverse DNA and RNA viruses](https://github.com/jiarong/VirSorter2)** ()
  Multi-classifier approach to detect diverse DNA and RNA virus sequences


### Phylogenetics <a id='tool-phylogenetics'></a>
- **[ViralMSA: massively scalable reference-guided multiple sequence alignment of viral genomes](https://github.com/niemasd/ViralMSA)** (multiple-sequence-alignment, viral-genomes, rna-viruses)
  Massively scalable reference-guided multiple sequence alignment of viral genomes, including RNA viruses


### Deep Learning <a id='tool-deep-learning'></a>
- **[Identifying viruses from metagenomic data using deep learning](https://github.com/jessieren/DeepVirFinder)** ()
  Deep learning method for identifying viral sequences from metagenomic data


- **[Artificial intelligence redefines RNA virus discovery](https://github.com/alibaba/LucaProt)** (Transformers)
  A deep learning-based tool that uses a self-supervised model for large-scale analysis of viral genomes, allowing for the discovery and characterization of new RNA viruses.


### Metagenomic Input <a id='tool-metagenomic-input'></a>
- **[VirBot: an RNA viral contig detector for metagenomic data](https://github.com/GreyGuoweiChen/VirBot)** ()
  A tool designed for detecting RNA viral contigs within metagenomic datasets.


### Genome-Annotation <a id='tool-genome-annotation'></a>
- **[VIGOR, an annotation program for small viral genomes](https://github.com/JCVI-VIRIFX/VIGOR4)** (orf-prediction, viral-genomes, rna-viruses)
  Viral genome ORF reader for annotation of viral genomes, including RNA viruses


### Assembly <a id='tool-assembly'></a>
- **[coronaSPAdes: from biosynthetic gene clusters to RNA viral assemblies](https://github.com/ablab/spades)** (coronavirus)
  Specialized assembler for coronavirus genomes and other RNA viral sequences


### Serratus <a id='tool-serratus'></a>
- **[palmscan: Ribovirus classification by a polymerase barcode sequence](https://github.com/rcedgar/palmscan)** ()
  A tool for classifying viral polymerase barcode sequences using the palmdb database, identifying operational taxonomic units (sOTUs) based on sequence similarity.


### Characterization <a id='tool-characterization'></a>
- **[VirID: Beyond Virus Discovery - An Integrated Platform for Comprehensive RNA Virus Characterization](https://github.com/ZiyueYang01/VirID)** ()
  A comprehensive platform for RNA virus characterization from transcriptomic data. It supports contig assembly, taxonomic classification, and variant analysis.


### Annotation <a id='tool-annotation'></a>
- **[VIRify: An integrated detection, annotation and taxonomic classification pipeline using virus-specific protein profile hidden Markov models](https://github.com/EBI-Metagenomics/emg-viral-pipeline)** (taxonomy)
  An integrated pipeline for virus detection, annotation, and taxonomic classification using hidden Markov models (HMMs) of viral-specific proteins.


### Genome Reconstruction <a id='tool-genome-reconstruction'></a>
- **[Omnifluss](https://github.com/rki-mf1/omnifluss)** (influenza, rna-viruses, workflow, genomic surveillance, NGS)
  Computational workflow for virus genome reconstruction at population-scale



## Databases <a id='database'></a>
### Database <a id='database-database'></a>
- **[NeoRdRp2 with improved seed data, annotations, and scoring](https://github.com/shoichisakaguchi/NeoRdRp)** (pHMM)
  An improved database and hidden Markov model (pHMM) dataset for identifying and classifying RNA-dependent RNA polymerase (RdRp) sequences.


- **[NeoRdRp: A Comprehensive Dataset for Identifying RNA-dependent RNA Polymerases of Various RNA Viruses from Metatranscriptomic Data](https://github.com/shoichisakaguchi/NeoRdRp)** (pHMM)
  A comprehensive dataset for identifying RNA-dependent RNA polymerases (RdRp) of various RNA viruses from metatranscriptomic data.


- **[RdRp-scan: A Bioinformatic Resource to Identify and Annotate Divergent RNA Viruses in Metagenomic Sequence Data](https://github.com/JustineCharon/RdRp-scan/)** (pHMM)
  A bioinformatic resource, including a curated dataset and HMM profiles, for identifying and annotating diverse RNA viruses in metagenomic data.


### Serratus <a id='database-serratus'></a>
- **[palmdb: Ribovirus classification by a polymerase barcode sequence](https://github.com/ababaian/palmdb)** ()
  A database containing over 145,000 viral polymerase palmprint (barcode) sequences, classified by clustering into species-like operational taxonomic units (sOTUs).



## Papers <a id='paper'></a>
### Fish <a id='paper-fish'></a>
<details>
<summary>**Meta-Transcriptomic Identification of Divergent Amnoonviridae in Fish**</summary>

Tilapia lake virus (TiLV) has caused mass mortalities in farmed and wild tilapia with serious economic and ecological consequences. Until recently, this virus was the sole member of the Amnoonviridae, a family within the order Articulavirales comprising segmented negative-sense RNA viruses. We sought to identify additional viruses within the Amnoonviridae through total RNA sequencing (meta-transcriptomics) and data mining of published transcriptomes. Accordingly, we sampled marine fish species from both Australia and China and discovered several segments of two new viruses within the Amnoonviridae, tentatively called Flavolineata virus and Piscibus virus, respectively. In addition, by mining vertebrate transcriptome data, we identified nine additional virus transcripts matching to multiple genomic segments of TiLV in both marine and freshwater fish. These new viruses retained sequence conservation with the distantly related Orthomyxoviridae in the RdRp subunit PB1, but formed a distinct and diverse phylogenetic group. These data suggest that the Amnoonviridae have a broad host range within fish and that greater animal sampling will identify additional divergent members of the Articulavirales

**DOI:** [10.3390/v12111254](https://doi.org/10.3390/v12111254)
**URL:** https://doi.org/10.3390/v12111254
**Topics:** fish
</details>


<details>
<summary>**Metagenomic sequencing reveals a lack of virus exchange between native and invasive freshwater fish across the Murray‚ÄìDarling Basin, Australi**</summary>

Biological invasions are among the biggest threats to freshwater biodiversity. This is increasingly relevant in the Murray‚ÄìDarling Basin, Australia, particularly since the introduction of the common carp (Cyprinus carpio). This invasive species now occupies up to ninety per cent of fish biomass, with hugely detrimental impacts on native fauna and flora. To address the ongoing impacts of carp, cyprinid herpesvirus 3 (CyHV-3) has been proposed as a potentially effective biological control agent. Crucially, however, it is unknown whether CyHV-3 and other cyprinid herpesviruses already exist in the Murray‚ÄìDarling. Further, little is known about those viruses that naturally occur in wild freshwater fauna, and the frequency with which these viruses jump species boundaries. To document the evolution and diversity of freshwater fish viromes and better understand the ecological context to the proposed introduction of CyHV-3, we performed a meta-transcriptomic viral survey of invasive and native fish across the Murray‚ÄìDarling Basin, covering over 2,200‚Äâkm of the river system. Across a total of thirty-six RNA libraries representing ten species, we failed to detect CyHV-3 nor any closely related viruses. Rather, meta-transcriptomic analysis identified eighteen vertebrate-associated viruses that could be assigned to the Arenaviridae, Astroviridae, Bornaviridae, Caliciviridae, Coronaviridae, Chuviridae, Flaviviridae, Hantaviridae, Hepeviridae, Paramyxoviridae, Picornaviridae, Poxviridae, Reoviridae and Rhabdoviridae families, and a further twenty-seven that were deemed to be associated with non-vertebrate hosts. Notably, we revealed a marked lack of viruses that are shared among invasive and native fish sampled here, suggesting that there is little virus transmission from common carp to native fish species, despite co-existing for over fifty years. Overall, this study provides the first data on the viruses naturally circulating in a major river system and supports the notion that fish harbour a large diversity of viruses with often deep evolutionary histories

**DOI:** [10.1093/ve/veab034](https://doi.org/10.1093/ve/veab034)
**URL:** https://doi.org/10.1093/ve/veab034
**Topics:** fish, ecology
</details>


<details>
<summary>**Diversity, evolution, and emergence of fish viruses**</summary>

The production of aquatic animals has more than doubled over the last 50 years and is anticipated to continually increase. While fish are recognized as a valuable and sustainable source of nutrition, particularly in the context of human population growth and climate change, the rapid expansion of aquaculture coincides with the emergence of highly pathogenic viruses that often spread globally through aquacultural practices. Here, we provide an overview of the fish virome and its relevance for disease emergence, with a focus on the insights gained through metagenomic sequencing, noting potential areas for future study. In particular, we describe the diversity and evolution of fish viruses, for which the majority have no known disease associations, and demonstrate how viruses emerge in fish populations, most notably at an expanding domestic-wild interface. We also show how wild fish are a powerful and tractable model system to study virus ecology and evolution more broadly and can be used to identify the major factors that shape vertebrate viromes. Central to this is a process of virus-host co-divergence that proceeds over many millions of years, combined with ongoing cross-species virus transmission

**DOI:** [10.1128/jvi.00118-24](https://doi.org/10.1128/jvi.00118-24)
**URL:** https://journals.asm.org/doi/full/10.1128/jvi.00118-24
**Topics:** fish
</details>


<details>
<summary>**Host and geography impact virus diversity in New Zealand‚Äôs longfin and shortfin eels**</summary>

The fishing and aquaculture industry is vital for global food security, yet viral diseases can result in mass fish die-off events. Determining the viromes of traditionally understudied species, such as fish, enhances our understanding of the global virosphere and the factors that influence virome composition and disease emergence. Very little is known about the viruses present in New Zealand‚Äôs native fish species, including the shortfin eel (Anguilla australis) and the longfin eel (Anguilla dieffenbachii), both of which are fished culturally by MƒÅori (the indigenous population of New Zealand) and commercially. Through a total RNA metatranscriptomic analysis of longfin and shortfin eels across three different geographic locations in the South Island of New Zealand, we aimed to determine whether viruses had jumped between the two eel species and whether eel virome composition was impacted by life stage, species, and geographic location. We identified nine viral species spanning eight different families, thereby enhancing our understanding of eel virus diversity in New Zealand and the host range of these viral families. Viruses of the family Flaviviridae (genus Hepacivirus) were widespread and found in both longfin and shortfin eels, indicative of cross-species transmission or virus-host co-divergence. Notably, both host specificity and geographic location appeared to influence eel virome composition, highlighting the complex interaction between viruses, hosts, and their ecosystems. This study broadens our understanding of viromes in aquatic hosts and highlights the importance of gaining baseline knowledge of fish viral abundance and diversity, particularly in aquatic species that are facing population declines

**DOI:** [10.1007/s00705-024-06019-1](https://doi.org/10.1007/s00705-024-06019-1)
**URL:** https://link.springer.com/article/10.1007/s00705-024-06019-1
**Topics:** fish
</details>


### Arthropods <a id='paper-arthropods'></a>
<details>
<summary>**Virome analysis of Amblyomma americanum, Dermacentor variabilis, and Ixodes scapularis ticks reveals novel highly divergent vertebrate and invertebrate viruses**</summary>

A wide range of bacterial pathogens have been identified in ticks, yet the diversity of viruses in ticks is largely unexplored. In the United States,Amblyomma americanum,Dermacentor variabilis, andIxodes scapularisare among the principal tick species associated with pathogen transmission. We used high-throughput sequencing to characterize the viromes of these tick species and identified the presence of Powassan virus and eight novel viruses. These included the most divergent nairovirus described to date, two new clades of tick-borne phleboviruses, a mononegavirus, and viruses with similarity to plant and insect viruses. Our analysis revealed that ticks are reservoirs for a wide range of viruses and suggests that discovery and characterization of tick-borne viruses will have implications for viral taxonomy and may provide insight into tick-transmitted diseases.Ticks are implicated as vectors of a wide array of human and animal pathogens. To better understand the extent of tick-borne diseases, it is crucial to uncover the full range of microbial agents associated with ticks. Our current knowledge of the diversity of tick-associated viruses is limited, in part due to the lack of investigation of tick viromes. In this study, we examined the viromes of three tick species from the United States. We found that ticks are hosts to highly divergent viruses across several taxa, including ones previously associated with human disease. Our data underscore the diversity of tick-associated viruses and provide the foundation for further studies into viral etiology of tick-borne diseases

**DOI:** [10.1128/JVI.01858-14](https://doi.org/10.1128/JVI.01858-14)
**URL:** https://doi.org/10.1128/JVI.01858-14
**Topics:** Arthropods
</details>


<details>
<summary>**Identification of diverse arthropod associated viruses in native Australian fleas**</summary>

Identification of diverse arthropod associated viruses in native Australian fleas

**DOI:** [10.1016/j.virol.2019.07.010](https://doi.org/10.1016/j.virol.2019.07.010)
**URL:** https://doi.org/10.1016/j.virol.2019.07.010
**Topics:** Arthropods
</details>


<details>
<summary>**Metatranscriptomics Reveals the Diversity of the Tick Virome in Northwest China**</summary>

Ticks are widely distributed in pastoral areas in northwestern China and act as vectors that carry and transmit a variety of pathogens, especially viruses. Our study revealed the diversity of tick viruses in Xinjiang and Inner Mongolia and uncovered the phylogenetic relationships of some RNA viruses, especially the important zoonotic tick-borne severe fever with thrombocytopenia syndrome virus in Inner Mongolia

**DOI:** [10.1128/spectrum.01115-22](https://doi.org/10.1128/spectrum.01115-22)
**URL:** https://doi.org/10.1128/spectrum.01115-22
**Topics:** Arthropods
</details>


<details>
<summary>**Extensive Diversity of RNA Viruses in Australian Ticks**</summary>

Each year a growing number of individuals along the east coast of Australia experience debilitating disease following tick bites. As there is no evidence for the presence of the causative agent of Lyme disease,Borrelia burgdorferisensu lato, in Australian ticks, the etiological basis of this disease syndrome remains controversial. To characterize the viruses associated with Australian ticks, particularly those that might be associated with mammalian infection, we performed unbiased RNA sequencing on 146 ticks collected across two locations along the coast of New South Wales, Australia. This revealed 19 novel RNA viruses from a diverse set of families. Notably, three of these viruses clustered with known mammalian viruses, including a novel coltivirus that was related to the human pathogen Colorado tick fever virus

**DOI:** [10.1128/JVI.01358-18](https://doi.org/10.1128/JVI.01358-18)
**URL:** https://doi.org/10.1128/JVI.01358-18
**Topics:** Arthropods
</details>


<details>
<summary>**Substantial viral and bacterial diversity at the bat‚Äìtick interface**</summary>

Ticks harbour a high diversity of viruses, bacteria and protozoa. The soft tick (Argasidae) is a common ectoparasite of bats in the Palearctic region and is suspected to be vector and reservoir of viruses and other microbial species in bat populations, some of which may act as zoonotic agents for human disease. The Soprano pipistrelle (Pipistrellus pygmaeus, Vespertilionidae) is widely distributed in Europe, where it can be found inside or close to human habitation. We used meta-transcriptomic sequencing to determine the RNA virome and common microbiota in blood-fed C. vespertilionis ticks collected from a Soprano pipistrelle bat roosting site in south-central Sweden. Our analyses identified 16 viruses from 11 virus families, of which 15 viruses were novel. For the first time in Sweden we identified Issuk-Kul virus, a zoonotic arthropod-borne virus previously associated with outbreaks of acute febrile illness in humans. Probable bat-associated and tick-borne viruses were classified within the families Nairoviridae, Caliciviridae and Hepeviridae, while other invertebrate-associated viruses included members of the Dicistroviridae, Iflaviridae, Nodaviridae, Partitiviridae, Permutotetraviridae, Polycipiviridae and Solemoviridae. Similarly, we found abundant bacteria in C. vespertilionis, including genera with known tick-borne bacteria, such as           Coxiella                spp. and           Rickettsia                spp. These findings demonstrate the remarkable diversity of RNA viruses and bacteria present in C. vespertilionis and highlight the importance of bat-associated ectoparasite surveillance as an effective and non-invasive means to track viruses and bacteria circulating in bats and ticks

**DOI:** [10.1099/mgen.0.000942](https://doi.org/10.1099/mgen.0.000942)
**URL:** https://www.microbiologyresearch.org/content/journal/mgen/10.1099/mgen.0.000942
**Topics:** Arthropods
</details>


<details>
<summary>**Unmapped RNA Virus Diversity in Termites and their Symbionts**</summary>

Despite their ecological importance, nothing is known about the diversity and abundance of RNA viruses in termites (Termitoidae). We used a metatranscriptomics approach to determine the RNA virome structure of 50 diverse species of termite that differ in both phylogenetic position and colony composition. From these samples, we identified 67 novel RNA viruses, characterized their genomes, quantified their abundance and inferred their evolutionary history. These viruses were found within or similar to those from the Togaviridae, Iflaviridae, Polycipiviridae, Flaviviridae, Leviviridae, Narnaviridae, Mitoviridae, Lispivirdae, Phasmaviridae, Picobirnaviridae and Partitiviridae. However, all viruses identified were novel and divergent, exhibiting only 20% to 45% amino acid identity to previously identified viruses. Our analysis suggested that 17 of the viruses identified were termite-infecting, with the remainder likely associated with the termite microbiome or diet. Unclassified sobemo-like and bunya-like viruses dominated termite viromes, while most of the phylogenetic diversity was provided by the picobirna- and mitovirus-like viruses. Of note was the identification of a novel flavi-like virus most closely related to those found in marine vertebrates and invertebrates. Notably, the sampling procedure had the strongest association with virome composition, with greater RNA virome diversity in libraries prepared from whole termite bodies than those that only sampled heads

**DOI:** [10.3390/v12101145](https://doi.org/10.3390/v12101145)
**URL:** https://doi.org/10.3390/v12101145
**Topics:** Arthropods
</details>


<details>
<summary>**Sustained RNA virome diversity in Antarctic penguins and their ticks**</summary>

Despite its isolation and extreme climate, Antarctica is home to diverse fauna and associated microorganisms. It has been proposed that the most iconic Antarctic animal, the penguin, experiences low pathogen pressure, accounting for their disease susceptibility in foreign environments. There is, however, a limited understanding of virome diversity in Antarctic species, the extent of in situ virus evolution, or how it relates to that in other geographic regions. To assess whether penguins have limited microbial diversity we determined the RNA viromes of three species of penguins and their ticks sampled on the Antarctic peninsula. Using total RNA sequencing we identified 107 viral species, comprising likely penguin associated viruses (n = 13), penguin diet and microbiome associated viruses (n = 82), and tick viruses (n = 8), two of which may have the potential to infect penguins. Notably, the level of virome diversity revealed in penguins is comparable to that seen in Australian waterbirds, including many of the same viral families. These data run counter to the idea that penguins are subject to lower pathogen pressure. The repeated detection of specific viruses in Antarctic penguins also suggests that rather than being simply spill-over hosts, these animals may act as key virus reservoirs

**DOI:** [10.1038/s41396-020-0643-1](https://doi.org/10.1038/s41396-020-0643-1)
**URL:** https://doi.org/10.1038/s41396-020-0643-1
**Topics:** Arthropods, Aves
</details>


<details>
<summary>**The virome of Rhipicephalus, Dermacentor and Haemaphysalis ticks from Eastern Romania includes novel viruses with potential relevance for public health**</summary>

The virome ofRhipicephalus,DermacentorandHaemaphysalisticks from Eastern Romania includes novel viruses with potential relevance for public health

**DOI:** [10.1111/tbed.14105](https://doi.org/10.1111/tbed.14105)
**URL:** https://doi.org/10.1111/tbed.14105
**Topics:** Arthropods, ticks
</details>


### Taxonomy <a id='paper-taxonomy'></a>
<details>
<summary>**Known phyla dominate the Tara Oceans RNA virome**</summary>

A recent study proposed five new RNA virus phyla, two of which, ‚ÄòTaraviricota‚Äô and ‚ÄòArctiviricota‚Äô, were stated to be ‚Äòdominant in the oceans‚Äô. However, the study‚Äôs assignments classify 28,353 putative RdRp-containing contigs to known phyla but only 886 (2.8%) to the five proposed new phyla combined. I re-mapped the reads to the contigs, finding that known phyla also account for a large majority (93.8%) of reads according to the study‚Äôs classifications, and that contigs originally assigned to ‚ÄòArctiviricota‚Äô accounted for only a tiny fraction (0.01%) of reads from Arctic Ocean samples. Performing my own virus identification and classifications, I found that 99.95‚Äâper cent of reads could be assigned to known phyla. The most abundant species was Beihai picorna-like virus 34 (15% of reads), and the most abundant order-like cluster was classified as Picornavirales (45% of reads). Sequences in the claimed new phylum ‚ÄòPomiviricota‚Äô were placed inside a phylogenetic tree for established order Durnavirales with 100‚Äâper cent confidence. Moreover, two contigs assigned to the proposed phylum ‚ÄòTaraviricota‚Äô were found to have high-identity alignments to dinoflagellate proteins, tentatively identifying this group of RdRp-like sequences as deriving from non-viral transcripts. Together, these results comprehensively contradict the claim that new phyla dominate the data

**DOI:** [10.1093/ve/vead063](https://doi.org/10.1093/ve/vead063)
**URL:** https://doi.org/10.1093/ve/vead063
**Topics:** taxonomy, diversity
</details>


### Marine <a id='paper-marine'></a>
<details>
<summary>**Freshwater macrophytes harbor viruses representing all five major phyla of the RNA viral kingdom Orthornavirae**</summary>

Research on aquatic plant viruses is lagging behind that of their terrestrial counterparts. To address this knowledge gap, here we identified viruses associated with freshwater macrophytes, a taxonomically diverse group of aquatic phototrophs that are visible with the naked eye. We surveyed pooled macrophyte samples collected at four spring sites in Florida, USA through next generation sequencing of RNA extracted from purified viral particles. Sequencing efforts resulted in the detection of 156 freshwater macrophyte associated (FMA) viral contigs, 37 of which approximate complete genomes or segments. FMA viral contigs represent putative members from all five major phyla of the RNA viral kingdom Orthornavirae. Similar to viral types found in land plants, viral sequences identified in macrophytes were dominated by positive-sense RNA viruses. Over half of the FMA viral contigs were most similar to viruses reported from diverse hosts in aquatic environments, including phototrophs, invertebrates, and fungi. The detection of FMA viruses from orders dominated by plant viruses, namely Patatavirales and Tymovirales, indicate that members of these orders may thrive in aquatic hosts. PCR assays confirmed the presence of putative FMA plant viruses in asymptomatic vascular plants, indicating that viruses with persistent lifestyles are widespread in macrophytes. The detection of potato virus Y and oat blue dwarf virus in submerged macrophytes suggests that terrestrial plant viruses infect underwater plants and highlights a potential terrestrial-freshwater plant virus continuum. Defining the virome of unexplored macrophytes will improve our understanding of virus evolution in terrestrial and aquatic primary producers and reveal the potential ecological impacts of viral infection in macrophytes

**DOI:** [10.7717/peerj.13875](https://doi.org/10.7717/peerj.13875)
**URL:** https://doi.org/10.7717/peerj.13875
**Topics:** marine
</details>


<details>
<summary>**Doubling of the known set of RNA viruses by metagenomic analysis of an aquatic virome**</summary>

RNA viruses in aquatic environments remain poorly studied. Here, we analysed the RNA virome from approximately 10‚Äâl water from Yangshan Deep-Water Harbour near the Yangtze River estuary in China and identified more than 4,500 distinct RNA viruses, doubling the previously known set of viruses. Phylogenomic analysis identified several major lineages, roughly, at the taxonomic ranks of class, order and family. The 719-member-strong Yangshan virus assemblage is the sister clade to the expansive class Alsuviricetes and consists of viruses with simple genomes that typically encode only RNA-dependent RNA polymerase (RdRP), capping enzyme and capsid protein. Several clades within the Yangshan assemblage independently evolved domain permutation in the RdRP. Another previously unknown clade shares ancestry with Potyviridae, the largest known plant virus family. The ‚ÄòAquatic picorna-like viruses/Marnaviridae‚Äô clade was greatly expanded, with more than 800 added viruses. Several RdRP-linked protein domains not previously detected in any RNA viruses were identified, such as the small ubiquitin-like modifier (SUMO) domain, phospholipase A2 and PrsW-family protease domain. Multiple viruses utilize alternative genetic codes implying protist (especially ciliate) hosts. The results reveal a vast RNA virome that includes many previously unknown groups. However, phylogenetic analysis of the RdRPs supports the previously established five-branch structure of the RNA virus evolutionary tree, with no additional phyla

**DOI:** [10.1038/s41564-020-0755-4](https://doi.org/10.1038/s41564-020-0755-4)
**URL:** https://doi.org/10.1038/s41564-020-0755-4
**Topics:** marine, phylogeny
</details>


<details>
<summary>**Ecological connectivity shapes quasispecies structure of RNA viruses in an Antarctic lake**</summary>

RNA viruses exist as complex mixtures of genotypes, known as quasispecies, where the evolution potential resides in the whole community of related genotypes. Quasispecies structure and dynamics have been studied in detail for virus infecting animals and plants but remain unexplored for those infecting micro‚Äêorganisms in environmental samples. We report the first metagenomic study of RNA viruses in an Antarctic lake (Lake Limnopolar, Livingston Island). Similar to low‚Äêlatitude aquatic environments, this lake harbours an RNA virome dominated by positive single‚Äêstrand RNA viruses from the order Picornavirales probably infecting micro‚Äêorganisms. Antarctic picorna‚Äêlike virus 1 (APLV1), one of the most abundant viruses in the lake, does not incorporate any mutation in the consensus sequence from 2006 to 2010 and shows stable quasispecies with low‚Äêcomplexity indexes. By contrast, APLV2‚ÄêAPLV3 are detected in the lake water exclusively in summer samples and are major constituents of surrounding cyanobacterial mats. Their quasispecies exhibit low complexity in cyanobacterial mat, but their run‚Äêoff‚Äêmediated transfer to the lake results in a remarkable increase of complexity that may reflect the convergence of different viral quasispecies from the catchment area or replication in a more diverse host community. This is the first example of viral quasispecies from natural aquatic ecosystems and points to ecological connectivity as a modulating factor of quasispecies complexity

**DOI:** [10.1111/mec.13321](https://doi.org/10.1111/mec.13321)
**URL:** https://doi.org/10.1111/mec.13321
**Topics:** marine
</details>


<details>
<summary>**Cryptic and abundant marine viruses at the evolutionary origins of Earth's RNA virome**</summary>

Whereas DNA viruses are known to be abundant, diverse, and commonly key ecosystem players, RNA viruses are insufficiently studied outside disease settings. In this study, we analyzed ‚âà28 terabases of Global Ocean RNA sequences to expand Earth‚Äôs RNA virus catalogs and their taxonomy, investigate their evolutionary origins, and assess their marine biogeography from pole to pole. Using new approaches to optimize discovery and classification, we identified RNA viruses that necessitate substantive revisions of taxonomy (doubling phyla and adding >50% new classes) and evolutionary understanding. ‚ÄúSpecies‚Äù-rank abundance determination revealed that viruses of the new phyla ‚ÄúTaraviricota,‚Äù a missing link in early RNA virus evolution, and ‚ÄúArctiviricota‚Äù are widespread and dominant in the oceans. These efforts provide foundational knowledge critical to integrating RNA viruses into ecological and epidemiological models

**DOI:** [10.1126/science.abm5847](https://doi.org/10.1126/science.abm5847)
**URL:** https://doi.org/10.1126/science.abm5847
**Topics:** marine
</details>


<details>
<summary>**Viromes in marine ecosystems reveal remarkable invertebrate RNA virus diversity**</summary>

Viromes in marine ecosystems reveal remarkable invertebrate RNA virus diversity

**DOI:** [10.1007/s11427-020-1936-2](https://doi.org/10.1007/s11427-020-1936-2)
**URL:** https://doi.org/10.1007/s11427-020-1936-2
**Topics:** marine
</details>


<details>
<summary>**Hidden diversity and evolution of viruses in market fish**</summary>

Hidden diversity and evolution of viruses in market fish

**DOI:** [10.1093/ve/vey031](https://doi.org/10.1093/ve/vey031)
**URL:** https://doi.org/10.1093/ve/vey031
**Topics:** marine, fish
</details>


<details>
<summary>**High diversity of unknown picorna-like viruses in the sea**</summary>

High diversity of unknown picorna-like viruses in the sea

**DOI:** [10.1038/nature01886](https://doi.org/10.1038/nature01886)
**URL:** https://doi.org/10.1038/nature01886
**Topics:** marine
</details>


<details>
<summary>**Virome composition in marine fish revealed by meta-transcriptomics**</summary>

Revealing the determinants of virome composition is central to placing disease emergence in a broader evolutionary context. Fish are the most species-rich group of vertebrates and so provide an ideal model system to study the factors that shape virome compositions and their evolution. We characterized the viromes of nineteen wild-caught species of marine fish using total RNA sequencing (meta-transcriptomics) combined with analyses of sequence and protein structural homology to identify divergent viruses that often evade characterization. From this, we identified twenty-five new vertebrate-associated viruses and a further twenty-two viruses likely associated with fish diet or their microbiomes. The vertebrate-associated viruses identified here included the first fish virus in the Matonaviridae (single-strand, positive-sense RNA virus). Other viruses fell within the Astroviridae, Picornaviridae, Arenaviridae, Reoviridae, Hepadnaviridae, Paramyxoviridae, Rhabdoviridae, Hantaviridae, Filoviridae, and Flaviviridae, and were sometimes phylogenetically distinct from known fish viruses. We also show how key metrics of virome composition‚Äîviral richness, abundance, and diversity‚Äîcan be analysed along with host ecological and biological factors as a means to understand virus ecology. Accordingly, these data suggest that that the vertebrate-associated viromes of the fish sampled here are predominantly shaped by the phylogenetic history (i.e. taxonomic order) of their hosts, along with several biological factors including water temperature, habitat depth, community diversity and swimming behaviour. No such correlations were found for viruses associated with porifera, molluscs, arthropods, fungi, and algae, that are unlikely to replicate in fish hosts. Overall, these data indicate that fish harbour particularly large and complex viromes and the vast majority of fish viromes are undescribed

**DOI:** [10.1093/ve/veab005](https://doi.org/10.1093/ve/veab005)
**URL:** https://doi.org/10.1093/ve/veab005
**Topics:** marine, fish
</details>


<details>
<summary>**Metagenomic analysis of coastal RNA virus communities**</summary>

RNA viruses infect marine organisms from bacteria to whales, but RNA virus communities in the sea remain essentially unknown. Reverse-transcribed whole-genome shotgun sequencing was used to characterize the diversity of uncultivated marine RNA virus assemblages. A diverse assemblage of RNA viruses, including a broad group of marine picorna-like viruses, and distant relatives of viruses infecting arthropods and higher plants were found. Communities were dominated by distinct genotypes with small genome sizes, and we completely assembled the genomes of several hitherto undiscovered viruses. Our results show that the oceans are a reservoir of previously unknown RNA viruses

**DOI:** [10.1126/science.1127404](https://doi.org/10.1126/science.1127404)
**URL:** https://doi.org/10.1126/science.1127404
**Topics:** marine
</details>


### Phages <a id='paper-phages'></a>
<details>
<summary>**Leviviricetes: expanding and restructuring the taxonomy of bacteria-infecting single-stranded RNA viruses**</summary>

The vast majority of described prokaryotic viruses have double-stranded or single-stranded DNA or double-stranded RNA genomes. Until 2020, a mere four prokaryotic single-stranded, positive-sense RNA viruses have been classified in two genera (Riboviria; Lenarviricota; Allassoviricetes; Leviviridae). Several recent metagenomic and metatranscriptomic studies revealed a vastly greater diversity of these viruses in prokaryotic soil communities than ever anticipated. Phylogenetic analysis of these newly discovered viruses prompted the reorganization of class Allassoviricetes, now renamed Leviviricetes, to include two orders, Norzivirales and Timlovirales, and a total of six families, 428 genera and 882 species. Here we outline the new taxonomy of Leviviricetes, approved and ratified in 2021 by the International Committee on Taxonomy of Viruses, and describe open-access hidden Markov models to accommodate the anticipated identification and future classification of hundreds, if not thousands, of additional class members into this new taxonomic framework

**DOI:** [10.1099/mgen.0.000686](https://doi.org/10.1099/mgen.0.000686)
**URL:** https://doi.org/10.1099/mgen.0.000686
**Topics:** phages
</details>


<details>
<summary>**Hyperexpansion of RNA Bacteriophage Diversity**</summary>

Hyperexpansion of RNA Bacteriophage Diversity

**DOI:** [10.1371/journal.pbio.1002409](https://doi.org/10.1371/journal.pbio.1002409)
**URL:** https://doi.org/10.1371/journal.pbio.1002409
**Topics:** Phages
</details>


<details>
<summary>**Expansion of the global RNA virome reveals diverse clades of bacteriophages**</summary>

Expansion of the global RNA virome reveals diverse clades of bacteriophages

**DOI:** [10.1016/j.cell.2022.08.023](https://doi.org/10.1016/j.cell.2022.08.023)
**URL:** https://doi.org/10.1016/j.cell.2022.08.023
**Topics:** phages
</details>


<details>
<summary>**Expansion of known ssRNA phage genomes: From tens to over a thousand**</summary>

We report a massive expansion in RNA-encoded bacterial viral genomes, which have been overlooked within environments

**DOI:** [10.1126/sciadv.aay5981](https://doi.org/10.1126/sciadv.aay5981)
**URL:** https://doi.org/10.1126/sciadv.aay5981
**Topics:** Phages
</details>


<details>
<summary>**Genomic and phylogenetic features of the Picobirnaviridae suggest microbial rather than animal hosts**</summary>

The RNA virus family Picobirnaviridae has traditionally been associated with the gastrointestinal systems of terrestrial mammals and birds, with the majority of viruses detected in animal stool samples. Metatranscriptomic studies of vertebrates, invertebrates, microbial communities, and environmental samples have resulted in an enormous expansion of the genomic and phylogenetic diversity of this family. Yet picobirnaviruses remain poorly classified, with only one genus and three species formally ratified by the International Committee of Virus Taxonomy. Additionally, an inability to culture picobirnaviruses in a laboratory setting or isolate them in animal tissue samples, combined with the presence of bacterial genetic motifs in their genomes, suggests that these viruses may represent RNA bacteriophage rather than being associated with animal infection. Utilising a data set of 2,286 picobirnaviruses sourced from mammals, birds, reptiles, fish, invertebrates, microbial communities, and environmental samples, we identified seven consistent phylogenetic clusters likely representing Picobirnavirus genera that we tentatively name ‚ÄòAlpha-‚Äô, ‚ÄòBeta-‚Äô, ‚ÄòGamma-‚Äô, ‚ÄòDelta-‚Äô, ‚ÄòEpsilon-‚Äô, ‚ÄòZeta-‚Äô, and ‚ÄòEtapicobirnavirus‚Äô. A statistical analysis of topological congruence between virus‚Äìhost phylogenies revealed more frequent cross-species transmission than any other RNA virus family. In addition, bacterial ribosomal binding site motifs were more enriched in Picobirnavirus genomes than in the two groups of established RNA bacteriophage‚Äîthe Leviviricetes and Cystoviridae. Overall, our findings support the hypothesis that the Picobirnaviridae have bacterial hosts and provide a lower-level taxonomic classification for this highly diverse and ubiquitous family of RNA viruses

**DOI:** [10.1093/ve/veae033](https://doi.org/10.1093/ve/veae033)
**URL:** https://academic.oup.com/ve/article/10/1/veae033/7655994?login=false
**Topics:** phages
</details>


### Aves <a id='paper-aves'></a>
<details>
<summary>**Metatranscriptomic Comparison of Viromes in Endemic and Introduced Passerines in New Zealand**</summary>

New Zealand/Aotearoa has many endemic passerine birds vulnerable to emerging infectious diseases. Yet little is known about viruses in passerines, and in some countries, including New Zealand, the virome of wild passerines has been only scarcely researched. Using metatranscriptomic sequencing we characterised the virome of New Zealand endemic and introduced species of passerine. Accordingly, we identified 34 possible avian viruses from cloacal swabs of 12 endemic and introduced bird species not showing signs of disease. These included a novel siadenovirus, iltovirus, and avastrovirus in the Eurasian blackbird (Turdus merula, an introduced species), song thrush (Turdus philomelos, introduced) and silvereye/tauhou (Zosterops lateralis, introduced), respectively. This is the first time novel viruses from these genera have been identified in New Zealand, likely reflecting prior undersampling. It also represents the first identification of an iltovirus and siadenovirus in blackbirds and thrushes globally. These three viruses were only found in introduced species and may pose a risk to endemic species if they were to jump species boundaries, particularly the iltoviruses and siadenoviruses that have a prior history of disease associations. Further virus study and surveillance are needed in New Zealand avifauna, particularly in Turdus populations and endemic species

**DOI:** [10.3390/v14071364](https://doi.org/10.3390/v14071364)
**URL:** https://doi.org/10.3390/v14071364
**Topics:** Aves
</details>


<details>
<summary>**RNA virome abundance and diversity is associated with host age in a bird species**</summary>

RNA virome abundance and diversity is associated with host age in a bird species

**DOI:** [10.1016/j.virol.2021.06.007](https://doi.org/10.1016/j.virol.2021.06.007)
**URL:** https://doi.org/10.1016/j.virol.2021.06.007
**Topics:** Aves
</details>


<details>
<summary>**Virome heterogeneity and connectivity in waterfowl and shorebird communities**</summary>

Models of host-microbe dynamics typically assume a single-host population infected by a single pathogen. In reality, many hosts form multi-species aggregations and may be infected with an assemblage of pathogens. We used a meta-transcriptomic approach to characterize the viromes of nine avian species in the Anseriformes (ducks) and Charadriiformes (shorebirds). This revealed the presence of 27 viral species, of which 24 were novel, including double-stranded RNA viruses (Picobirnaviridae and Reoviridae), single-stranded RNA viruses (Astroviridae, Caliciviridae, Picornaviridae), a retro-transcribing DNA virus (Hepadnaviridae), and a single-stranded DNA virus (Parvoviridae). These viruses comprise multi-host generalist viruses and those that are host-specific, indicative of both virome connectivity (host sharing) and heterogeneity (host specificity). Virome connectivity was apparent in two well described multi-host virus species -avian coronavirus and influenza A virus- and a novel Rotavirus species that were shared among some Anseriform species, while virome heterogeneity was reflected in the absence of viruses shared between Anseriformes and Charadriiformes, as well as differences in viral abundance and alpha diversity among species. Overall, we demonstrate complex virome structures across host species that co-exist in multi-species aggregations

**DOI:** [10.1038/s41396-019-0458-0](https://doi.org/10.1038/s41396-019-0458-0)
**URL:** https://doi.org/10.1038/s41396-019-0458-0
**Topics:** Aves
</details>


### Evolution <a id='paper-evolution'></a>
<details>
<summary>**Metagenomics reshapes the concepts of RNA virus evolution by revealing extensive horizontal virus transfer**</summary>

Metagenomics reshapes the concepts of RNA virus evolution by revealing extensive horizontal virus transfer

**DOI:** [10.1016/j.virusres.2017.10.020](https://doi.org/10.1016/j.virusres.2017.10.020)
**URL:** https://doi.org/10.1016/j.virusres.2017.10.020
**Topics:** evolution
</details>


### Phylogeny <a id='paper-phylogeny'></a>
<details>
<summary>**Global phylogenetic analysis of the RNA-dependent RNA polymerase with OrViT (OrthornaVirae Tree)**</summary>

Viruses of the kingdom Orthornavirae are the causative agents of many diseases in humans, animals and plants and play an important role in the ecology of the biosphere. Novel orthornaviral viral sequences are constantly being discovered from environmental datasets, but generating high-quality and comprehensive phylogenetic trees of Orthornavirae to resolve their taxonomic and phylogenetic relationships is still a challenge. To assist microbial ecologists and virologists with this task, we developed OrViT (OrthornaVirae Tree), a pipeline that integrates and updates published methods and bridges various public software to generate a global phylogenetic tree of the RNA-dependent RNA polymerase (RdRp) encoded by all orthornaviral genomes. The pipeline can infer the phylogenetic relationships between RdRp sequences extracted from the RefSeq viral database and the users‚Äô own assembled contigs or protein datasets. The results from OrViT can be used for the taxonomic identification of novel viruses and suggest revisions of the existing phylogeny of RNA viruses. OrViT includes several Perl and Bash scripts assembled into a Makefile, making it portable between different Linux-based operating systems and easy to use. OrViT is freely available from https://github.com/chengdongqiang/OrViT

**DOI:** [10.3389/fviro.2022.981177](https://doi.org/10.3389/fviro.2022.981177)
**URL:** https://doi.org/10.3389/fviro.2022.981177
**Topics:** phylogeny, tool
</details>


<details>
<summary>**Evidence for an ancient aquatic origin of the RNA viral order Articulavirales**</summary>

The emergence of previously unknown disease-causing viruses in mammals is in part the result of a long-term evolutionary process. Reconstructing the deep phylogenetic histories of viruses helps identify major evolutionary transitions and contextualizes the emergence of viruses in new hosts. We used a combination of total RNA sequencing and transcriptome data mining to extend the diversity and evolutionary history of the RNA virus orderArticulavirales, which includes the influenza viruses. We identified instances ofArticulaviralesin the invertebrate phylum Cnidaria (including corals), constituting a novel and divergent family that we provisionally named the ‚ÄúCnidenomoviridae.‚Äù We further extended the evolutionary history of the influenza virus lineage by identifying four divergent, fish-associated influenza-like viruses, thereby supporting the hypothesis that fish were among the first hosts of influenza viruses. In addition, we substantially expanded the phylogenetic diversity of quaranjaviruses and proposed that this genus be reclassified as a family‚Äîthe ‚ÄúQuaranjaviridae.‚Äù Within this putative family, we identified a novel arachnid-infecting genus, provisionally named ‚ÄúCheliceravirus.‚Äù Notably, we observed a close phylogenetic relationship between the Crustacea- and Chelicerata-infecting ‚ÄúQuaranjaviridae‚Äù that is inconsistent with virus‚Äìhost codivergence. Together, these data suggest that theArticulaviraleshas evolved over at least 600 million years, first emerging in aquatic animals. Importantly, the evolution of theArticulaviraleswas likely shaped by multiple aquatic‚Äìterrestrial transitions and substantial host jumps, some of which are still observable today

**DOI:** [10.1073/pnas.2310529120](https://doi.org/10.1073/pnas.2310529120)
**URL:** https://www.pnas.org/doi/abs/10.1073/pnas.2310529120
**Topics:** phylogeny, aquatic
</details>


<details>
<summary>**Polycipiviridae: a proposed new family of polycistronic picorna-like RNA viruses**</summary>

Solenopsis invicta virus 2 is a single-stranded positive-sense picorna-like RNA virus with an unusual genome structure. The monopartite genome of approximately 11‚Äâkb contains four open reading frames in its 5‚Ä≤ third, three of which encode proteins with homology to picornavirus-like jelly-roll fold capsid proteins. These are followed by an intergenic region, and then a single long open reading frame that covers the 3‚Ä≤ two-thirds of the genome. The polypeptide translation of this 3‚Ä≤ open reading frame contains motifs characteristic of picornavirus-like helicase, protease and RNA-dependent RNA polymerase domains. An inspection of public transcriptome shotgun assembly sequences revealed five related apparently nearly complete virus genomes isolated from ant species and one from a dipteran insect. By high-throughput sequencing and in silico assembly of RNA isolated from Solenopsis invicta and four other ant species, followed by targeted Sanger sequencing, we obtained nearly complete genomes for four further viruses in the group. Four further sequences were obtained from a recent large-scale invertebrate virus study. The 15 sequences are highly divergent (pairwise amino acid identities of as low as 17‚Ää% in the non-structural polyprotein), but possess the same overall polycistronic genome structure, which is distinct from all other characterized picorna-like viruses. Consequently, we propose the formation of a new virus family, Polycipiviridae, to classify this clade of arthropod-infecting polycistronic picorna-like viruses. We further propose that this family be divided into three genera: Chipolycivirus (2 species), Hupolycivirus (2 species) and Sopolycivirus (11 species), with members of the latter infecting ants in at least 3 different subfamilies

**DOI:** [10.1099/jgv.0.000902](https://doi.org/10.1099/jgv.0.000902)
**URL:** https://doi.org/10.1099/jgv.0.000902
**Topics:** phylogeny
</details>


<details>
<summary>**The discovery of three new hare lagoviruses reveals unexplored viral diversity in this genus**</summary>

The discovery of three new hare lagoviruses reveals unexplored viral diversity in this genus

**DOI:** [10.1093/ve/vez005](https://doi.org/10.1093/ve/vez005)
**URL:** https://doi.org/10.1093/ve/vez005
**Topics:** phylogeny
</details>


<details>
<summary>**The evolutionary history of vertebrate RNA viruses**</summary>

The evolutionary history of vertebrate RNA viruses

**DOI:** [10.1038/s41586-018-0012-7](https://doi.org/10.1038/s41586-018-0012-7)
**URL:** https://doi.org/10.1038/s41586-018-0012-7
**Topics:** phylogeny, vertebrate
</details>


### Mammalia <a id='paper-mammalia'></a>
<details>
<summary>**Comparative Analysis of RNA Virome Composition in Rabbits and Associated Ectoparasites**</summary>

Ectoparasites play an important role in the transmission of many vertebrate-infecting viruses, including Zika and dengue viruses. Although it is becoming increasingly clear that invertebrate species harbor substantial virus diversity, it is unclear how many of the viruses carried by invertebrates have the potential to infect vertebrate species. We used the European rabbit (Oryctolagus cuniculus) as a model species to compare virome compositions in a vertebrate host and known associated ectoparasite mechanical vectors, in this case, fleas and blowflies. In particular, we aimed to infer the extent of viral transfer between these distinct types of host. Our analysis revealed that despite extensive viral diversity in both rabbits and associated ectoparasites, and the close interaction of these vertebrate and invertebrate species, biological viral transmission from ectoparasites to vertebrate species is rare. We did, however, find evidence to support the idea of a role of blowflies in transmitting viruses without active replication in the insect

**DOI:** [10.1128/JVI.02119-19](https://doi.org/10.1128/JVI.02119-19)
**URL:** https://doi.org/10.1128/JVI.02119-19
**Topics:** Mammalia, ectoparasite
</details>


<details>
<summary>**Unrecognized diversity of mammalian orthoreoviruses in North American bats**</summary>

Unrecognized diversity of mammalian orthoreoviruses in North American bats

**DOI:** [10.1016/j.virol.2022.03.012](https://doi.org/10.1016/j.virol.2022.03.012)
**URL:** https://doi.org/10.1016/j.virol.2022.03.012
**Topics:** Mammalia, bats
</details>


<details>
<summary>**Host traits shape virome composition and virus transmission in wild small mammals**</summary>

Molecular basis of anaphylatoxin binding, activation, and signaling bias at complement receptors

**DOI:** [10.1016/j.cell.2023.09.020](https://doi.org/10.1016/j.cell.2023.09.020)
**URL:** https://doi.org/10.1016/j.cell.2023.09.020
**Topics:** Mammalia, ecology
</details>


<details>
<summary>**Identification of novel mammalian viruses in tree shrews (Tupaia belangeri chinensis)**</summary>

Identification of novel mammalian viruses in tree shrews (Tupaia belangeri chinensis)

**DOI:** [10.24272/j.issn.2095-8137.2023.306](https://doi.org/10.24272/j.issn.2095-8137.2023.306)
**URL:** https://www.zoores.ac.cn/en/article/doi/10.24272/j.issn.2095-8137.2023.306
**Topics:** Mammalia
</details>


<details>
<summary>**Virome analysis of New Zealand‚Äôs bats reveals cross-species viral transmission among the Coronaviridae**</summary>

The lesser short-tailed bat (Mystacina tuberculata) and the long-tailed bat (Chalinolobus tuberculatus) are Aotearoa New Zealand‚Äôs only native extant terrestrial mammals and are believed to have migrated from Australia. Long-tailed bats arrived in New Zealand an estimated two million years ago and are closely related to other Australian bat species. Lesser short-tailed bats, in contrast, are the only extant species within the Mystacinidae and are estimated to have been living in isolation in New Zealand for the past 16‚Äì18 million years. Throughout this period of isolation, lesser short-tailed bats have become one of the most terrestrial bats in the world. Through a metatranscriptomic analysis of guano samples from eight locations across New Zealand, we aimed to characterise the viromes of New Zealand‚Äôs bats and determine whether viruses have jumped between these species over the past two million years. High viral richness was observed among long-tailed bats with viruses spanning seven different viral families. In contrast, no bat-specific viruses were identified in lesser short-tailed bats. Both bat species harboured an abundance of likely dietary- and environment-associated viruses. We also identified alphacoronaviruses in long-tailed bat guano that had previously been identified in lesser short-tailed bats, suggesting that these viruses had jumped the species barrier after long-tailed bats migrated to New Zealand. Of note, an alphacoronavirus species discovered here possessed a complete genome of only 22,416 nucleotides with entire deletions or truncations of several non-structural proteins, thereby representing what may be the shortest genome within the Coronaviridae identified to date. Overall, this study has revealed a diverse range of novel viruses harboured by New Zealand‚Äôs only native terrestrial mammals, in turn expanding our understanding of bat viral dynamics and evolution globally

**DOI:** [10.1093/ve/veae008](https://doi.org/10.1093/ve/veae008)
**URL:** https://academic.oup.com/ve/article/10/1/veae008/7591602?login=false
**Topics:** Mammalia, bats, Coronaviridae
</details>


<details>
<summary>**Newly identified viral genomes in pangolins with fatal disease**</summary>

Epizootic pathogens pose a major threat to many wildlife species, particularly in the context of rapidly changing environments. Pangolins (order Pholidota) are highly threatened mammals, in large part due to the trade in illegal wildlife. During July to August 2018 four sick wild pangolins (three Manis javanica and one Manis pentadactyla) exhibiting a variety of clinical symptoms were rescued by the Jinhua Wildlife Protection Station in Zhejiang province, China. Although three of these animals died, fortunately one recovered after 2 weeks of symptomatic treatment. Using meta-transcriptomics combined with reverse transcription polymerase chain reaction (RT-PCR), we identified two novel RNA viruses in two of the dead pangolins. Genomic analysis revealed that these viruses were most closely related to pestiviruses and coltiviruses, although still highly genetically distinct, with more than 48 and 25 per cent sequence divergence at the amino acid level, respectively. We named these Dongyang pangolin virus (DYPV) and Lishui pangolin virus (LSPV) based on the sampling site and hosts. Although coltiviruses (LSPV) are known to be transmitted by ticks, we found no evidence of LSPV in ticks sampled close to where the pangolins were collected. In addition, although DYPV was present in nymph ticks (Amblyomma javanense) collected from a diseased pangolin, they were not found in the local tick population. Epidemiological investigation revealed that both novel viruses might have been imported following the illegal international trade of pangolins. Hence, these data indicate that illegal wildlife trafficking not only threatens the status of pangolin populations, but may also spread epizootic pathogens

**DOI:** [10.1093/ve/veaa020](https://doi.org/10.1093/ve/veaa020)
**URL:** https://doi.org/10.1093/ve/veaa020
**Topics:** Mammalia
</details>


<details>
<summary>**Metagenomic discovery and co-infection of diverse wobbly possum disease viruses and a novel hepacivirus in Australian brushtail possums**</summary>

Australian brushtail possums (Trichosurus vulpecula) are an introduced pest species in New Zealand, but native to Australia where they are protected for biodiversity conservation. Wobbly possum disease (WPD) is a fatal neurological disease of Australian brushtail possums described in New Zealand populations that has been associated with infection by the arterivirus (Arteriviridae) wobbly possum disease virus (WPDV-NZ). Clinically, WPD-infected possums present with chronic meningoencephalitis, choroiditis and multifocal neurological symptoms including ataxia, incoordination, and abnormal gait            We conducted a retrospective investigation to characterise WPD in native Australian brushtail possums, and used a bulk meta-transcriptomic approach (i.e. total RNA-sequencing) to investigate its potential viral aetiology. PCR assays were developed for case diagnosis and full genome recovery in the face of extensive genetic variation            We identified genetically distinct lineages of arteriviruses from archival tissues of WPD-infected possums in Australia, termed wobbly possum disease virus AU1 and AU2. Phylogenetically, WPDV-AU1 and WPDV-AU2 shared only ~‚Äâ70% nucleotide similarity to each other and the WPDV-NZ strain, suggestive of a relatively ancient divergence. Notably, we also identified a novel and divergent hepacivirus (Flaviviridae) - the first in a marsupial - in both WPD-infected and uninfected possums, indicative of virus co-infection        Conclusions    We have identified marsupial-specific lineages of arteriviruses in mainland Australia that are genetically distinct from that in New Zealand, in some cases co-infecting animals with a novel hepacivirus. Our study provides new insight into the hidden genetic diversity of arteriviruses, the capacity for virus co-infection, and highlights the utility of meta-transcriptomics for disease investigation in a One Health context

**DOI:** [10.1186/s42522-019-0006-x](https://doi.org/10.1186/s42522-019-0006-x)
**URL:** https://doi.org/10.1186/s42522-019-0006-x
**Topics:** Mammalia
</details>


<details>
<summary>**Red fox viromes in urban and rural landscapes**</summary>

The Red fox (Vulpes vulpes) has established large populations in Australia‚Äôs urban and rural areas since its introduction following European settlement. The cryptic and highly adaptable nature of foxes allows them to invade cities and live among humans whilst remaining largely unnoticed. Urban living and access to anthropogenic food resources also influence fox ecology. Urban foxes grow larger, live at higher densities, and are more social than their rural counterparts. These ecological changes in urban red foxes are likely to impact the pathogens that they harbour, and foxes could pose a disease risk to humans and other species that share these urban spaces. To investigate this possibility, we used a meta-transcriptomic approach to characterise the virome of urban and rural foxes across the Greater Sydney region in Australia. Urban and rural foxes differed significantly in virome composition, with rural foxes harbouring a greater abundance of viruses compared to their urban counterparts. We identified ten potentially novel vertebrate-associated viruses in both urban and rural foxes, some of which are related to viruses associated with disease in domestic species and humans. These included members of the Astroviridae, Picobirnaviridae, Hepeviridae, and Picornaviridae as well as rabbit haemorrhagic disease virus-2. This study sheds light on the viruses carried by urban and rural foxes and emphasises the need for greater genomic surveillance of foxes and other invasive species at the human‚Äìwildlife interface

**DOI:** [10.1093/ve/veaa065](https://doi.org/10.1093/ve/veaa065)
**URL:** https://doi.org/10.1093/ve/veaa065
**Topics:** Mammalia
</details>


<details>
<summary>**Discovery and Prevalence of Divergent RNA Viruses in European Field Voles and Rabbits**</summary>

The advent of unbiased metagenomic virus discovery has revolutionized studies of virus biodiversity and evolution. Despite this, our knowledge of the virosphere, including in mammalian species, remains limited. We used unbiased metagenomic sequencing to identify RNA viruses in European field voles and rabbits. Accordingly, we identified a number of novel RNA viruses including astrovirus, rotavirus A, picorna-like virus and a narmovirus (paramyxovirus). In addition, we identified a sobemovirus and a novel luteovirus that likely originated from the rabbit diet. These newly discovered viruses were often divergent from those previously described. The novel astrovirus was most closely related to a virus sampled from the rodent-eating European roller bird (Coracias garrulous). PCR screening revealed that the novel narmovirus in the UK field vole had a prevalence of approximately 4%, and shared common ancestry with other rodent narmoviruses sampled globally. Two novel rotavirus A sequences were detected in a UK field vole and a French rabbit, the latter with a prevalence of 5%. Finally, a highly divergent picorna-like virus found in the gut of the French rabbit virus was only ~35% similar to an arilivirus at the amino acid level, suggesting the presence of a novel viral genus within the Picornaviridae

**DOI:** [10.3390/v12010047](https://doi.org/10.3390/v12010047)
**URL:** https://doi.org/10.3390/v12010047
**Topics:** Mammalia
</details>


<details>
<summary>**Novel highly divergent reassortant bat rotaviruses in Cameroon, without evidence of zoonosis**</summary>

Bats are an important reservoir for zoonotic viruses. To date, only three RVA strains have been reported in bats in Kenya and China. In the current study we investigated the genetic diversity of RVAs in fecal samples from 87 straw-colored fruit bats living in close contact with humans in Cameroon using viral metagenomics. Five (near) complete RVA genomes were obtained. A single RVA strain showed a partial relationship with the Kenyan bat RVA strain, whereas the other strains were completely novel. Only the VP7 and VP4 genes showed significant variability, indicating the occurrence of frequent reassortment events. Comparing these bat RVA strains with currently used human RVA screening primers indicated that most of the novel VP7 and VP4 segments would not be detected in routine epidemiological screening studies. Therefore, novel consensus screening primers were developed and used to screen samples from infants with gastroenteritis living in close proximity with the studied bat population. Although RVA infections were identified in 36% of the infants, there was no evidence of zoonosis. This study identified multiple novel bat RVA strains, but further epidemiological studies in humans will have to assess if these viruses have the potential to cause gastroenteritis in humans

**DOI:** [10.1038/srep34209](https://doi.org/10.1038/srep34209)
**URL:** https://doi.org/10.1038/srep34209
**Topics:** Mammalia, bats
</details>


<details>
<summary>**Faecal virome of the Australian grey-headed flying fox from urban/suburban environments contains novel coronaviruses, retroviruses and sapoviruses**</summary>

Faecal virome of the Australian grey-headed flying fox from urban/suburban environments contains novel coronaviruses, retroviruses and sapoviruses

**DOI:** [10.1016/j.virol.2022.09.002](https://doi.org/10.1016/j.virol.2022.09.002)
**URL:** https://doi.org/10.1016/j.virol.2022.09.002
**Topics:** Mammalia
</details>


### Gastropods <a id='paper-gastropods'></a>
<details>
<summary>**Tracing RNA viruses associated with Nudibranchia gastropods**</summary>

Nudibranchia is an under-studied taxonomic group of gastropods, including more than 3,000 species with colourful and extravagant body shapes and peculiar predatory and defensive strategies. Although symbiosis with bacteria has been reported, no data are available for the nudibranch microbiome nor regarding viruses possibly associated with these geographically widespread species  MethodsBased on 47 available RNA sequencing datasets including more than two billion reads of 35 nudibranch species, a meta-transcriptome assembly was constructed. Taxonomic searches with DIAMOND, RNA-dependent-RNA-polymerase identification with palmscan and viral hallmark genes identification by VirSorter2 in combination with CheckV were applied to identify genuine viral genomes, which were then annotated using CAT  ResultsA total of 20 viral genomes were identified as bona fide viruses, among 552 putative viral contigs resembling both RNA viruses of the Negarnaviricota, Pisuviricota, Kitrinoviricota phyla and actively transcribing DNA viruses of the Cossaviricota and Nucleocytoviricota phyla. The 20 commonly identified viruses showed similarity with RNA viruses identified in other RNA-seq experiments and can be putatively associated with bacteria, plant and arthropod hosts by co-occurence analysis. The RNA samples having the highest viral abundances showed a heterogenous and mostly sample-specific distribution of the identified viruses, suggesting that nudibranchs possess diversified and mostly unknown viral communities

**DOI:** [10.7717/peerj.13410](https://doi.org/10.7717/peerj.13410)
**URL:** https://doi.org/10.7717/peerj.13410
**Topics:** gastropods
</details>


### Insects <a id='paper-insects'></a>
<details>
<summary>**Draft Genome Sequences of Six Novel Picorna-Like Viruses from Washington State Spiders**</summary>

We report draft genome sequences of six novelPicornaviralesmembers from six different spider species found in Washington state. These six viral sequences distinctly clustered together phylogenetically with less than 35% amino acid identity to the closest reference viral genome

**DOI:** [10.1128/genomeA.01705-16](https://doi.org/10.1128/genomeA.01705-16)
**URL:** https://doi.org/10.1128/genomeA.01705-16
**Topics:** Insects
</details>


<details>
<summary>**Viromics of extant insect orders unveil the evolution of the flavi-like superfamily**</summary>

Insects are the most diversified and species-rich group of animals and harbor an immense diversity of viruses. Several taxa in the flavi-like superfamily, such as the genus Flavivirus, are associated with insects; however, systematic studies on insect virus genetic diversity are lacking, limiting our understanding of the evolution of the flavi-like superfamily. Here, we examined the diversity of flavi-like viruses within the most complete and up-to-date insect transcriptome collection comprising 1,243 insect species by employing a Flaviviridae RdRp profile hidden Markov model search. We identified seventy-six viral sequences in sixty-one species belonging to seventeen insect, one entognathan, and one arachnidan orders. Phylogenetic analyses revealed that twenty-seven sequences fell within the Flaviviridae phylogeny but did not group with established genera. Despite the large diversity of insect hosts studied, we only detected one virus in a blood-feeding insect, which branched within the genus Flavivirus, indicating that this genus likely diversified only in hematophagous arthropods. Nine new jingmenviruses with novel host associations were identified. One of the jingmenviruses established a deep rooting lineage additional to the insect- and tick-associated clades. Segment co-segregation phylogenies support the separation of tick- and insect-associated groups within jingmenviruses, with evidence for segment reassortment. In addition, fourteen viruses grouped with unclassified flaviviruses encompassing genome length of up to 20‚Äâkb. Species-specific clades for Hymenopteran- and Orthopteran-associated viruses were identified. Forty-nine viruses populated three highly diversified clades in distant relationship to Tombusviridae, a plant-infecting virus family, suggesting the detection of three previously unknown insect-associated families that contributed to tombusvirus evolution

**DOI:** [10.1093/ve/veab030](https://doi.org/10.1093/ve/veab030)
**URL:** https://doi.org/10.1093/ve/veab030
**Topics:** Insects
</details>


<details>
<summary>**Re-assessing the diversity of negative strand RNA viruses in insects**</summary>

Re-assessing the diversity of negative strand RNA viruses in insects

**DOI:** [10.1371/journal.ppat.1008224](https://doi.org/10.1371/journal.ppat.1008224)
**URL:** https://doi.org/10.1371/journal.ppat.1008224
**Topics:** Insects
</details>


### Parasite <a id='paper-parasite'></a>
<details>
<summary>**A parasite odyssey: An RNA virus concealed in Toxoplasma gondii**</summary>

We are entering a ‚ÄòPlatinum Age of Virus Discovery‚Äô, an era marked by exponential growth in the discovery of virus biodiversity, and driven by advances in metagenomics and computational analysis. In the ecosystem of a human (or any animal) there are more species of viruses than simply those directly infecting the animal cells. Viruses can infect all organisms constituting the microbiome, including bacteria, fungi, and unicellular parasites. Thus the complexity of possible interactions between host, microbe, and viruses is unfathomable. To understand this interaction network we must employ computationally assisted virology as a means of analyzing and interpreting the millions of available samples to make inferences about the ways in which viruses may intersect human health. From a computational viral screen of human neuronal datasets, we identified a novel narnavirus Apocryptovirus odysseus (Ao) which likely infects the neurotropic parasite Toxoplasma gondii. Previously, several parasitic protozoan viruses (PPVs) have been mechanistically established as triggers of host innate responses, and here we present in silico evidence that Ao is a plausible pro-inflammatory factor in human and mouse cells infected by T. gondii. T. gondii infects billions of people worldwide, yet the prognosis of toxoplasmosis disease is highly variable, and PPVs like Ao could function as a hitherto undescribed hypervirulence factor. In a broader screen of over 7.6 million samples, we explored phylogenetically proximal viruses to Ao and discovered nineteen Apocryptovirus species, all found in libraries annotated as vertebrate transcriptome or metatranscriptomes. While samples containing this genus of narnaviruses are derived from sheep, goat, bat, rabbit, chicken, and pigeon samples, the presence of virus is strongly predictive of parasitic Apicomplexa nucleic acid co-occurrence, supporting the fact that Apocryptovirus is a genus of parasite-infecting viruses. This is a computational proof-of-concept study in which we rapidly analyze millions of datasets from which we distilled a mechanistically, ecologically, and phylogenetically refined hypothesis. We predict that this highly diverged Ao RNA virus is biologically a T. gondii infection, and that Ao, and other viruses like it, will modulate this disease which afflicts billions worldwide

**DOI:** [10.1093/ve/veae040](https://doi.org/10.1093/ve/veae040)
**URL:** https://doi.org/10.1093/ve/veae040
**Topics:** parasite, protozoa
</details>


<details>
<summary>**A World of Viruses Nested within Parasites: Unraveling Viral Diversity within Parasitic Flatworms (Platyhelminthes)**</summary>

Little is known about the diversity of parasite-associated viruses and how these viruses may impact parasite fitness, parasite-host interactions, and virus evolution. The discovery of over a hundred viruses associated with a range of free-living and parasitic flatworms, including parasites of economic and clinical relevance, allowed us to compare the viromes of flatworms with contrasting lifestyles

**DOI:** [10.1128/spectrum.00138-22](https://doi.org/10.1128/spectrum.00138-22)
**URL:** https://doi.org/10.1128/spectrum.00138-22
**Topics:** parasite
</details>


### Mosquito <a id='paper-mosquito'></a>
<details>
<summary>**Metagenomic analysis of individual mosquito viromes reveals the geographical patterns and drivers of viral diversity**</summary>

Metagenomic analysis of individual mosquito viromes reveals the geographical patterns and drivers of viral diversity

**DOI:** [10.1038/s41559-024-02365-0](https://doi.org/10.1038/s41559-024-02365-0)
**URL:** https://www.nature.com/articles/s41559-024-02365-0
**Topics:** Mosquito
</details>


<details>
<summary>**An insight into the sialotranscriptome and virome of Amazonian anophelines**</summary>

An insight into the sialotranscriptome and virome of Amazonian anophelines

**DOI:** [10.1186/s12864-019-5545-0](https://doi.org/10.1186/s12864-019-5545-0)
**URL:** https://doi.org/10.1186/s12864-019-5545-0
**Topics:** Mosquito
</details>


<details>
<summary>**Stable distinct core eukaryotic viromes in different mosquito species from Guadeloupe, using single mosquito viral metagenomics**</summary>

Stable distinct core eukaryotic viromes in different mosquito species from Guadeloupe, using single mosquito viral metagenomics

**DOI:** [10.1186/s40168-019-0734-2](https://doi.org/10.1186/s40168-019-0734-2)
**URL:** https://doi.org/10.1186/s40168-019-0734-2
**Topics:** Mosquito
</details>


<details>
<summary>**High-Resolution Metatranscriptomics Reveals the Ecological Dynamics of Mosquito-Associated RNA Viruses in Western Australia**</summary>

Mosquitoes harbor a high diversity of RNA viruses, including many that impact human health. Despite a growing effort to describe the extent and nature of the mosquito virome, little is known about how these viruses persist, spread, and interact with both their hosts and other microbes. To address this issue we performed a metatranscriptomics analysis of 12 Western Australian mosquito populations structured by species and geographic location. Our results identified the complete genomes of 24 species of RNA viruses from a diverse range of viral families and orders, among which 19 are newly described. Comparisons of viromes revealed a striking difference between the two mosquito genera, with viromes of mosquitoes of theAedesgenus exhibiting substantially less diversity and lower abundances than those of mosquitoes of theCulexgenus, within which the viral abundance reached 16.87% of the total non-rRNA. In addition, there was little overlap in viral diversity between the two genera, although the viromes were very similar among the threeCulexspecies studied, suggesting that the host taxon plays a major role in structuring virus diversity. In contrast, we found no evidence that geographic location played a major role in shaping RNA virus diversity, and several viruses discovered here exhibited high similarity (95 to 98% nucleotide identity) to those from Indonesia and China. Finally, using abundance-level and phylogenetic relationships, we were able to distinguish potential mosquito viruses from those present in coinfecting bacteria, fungi, and protists. In sum, our metatranscriptomics approach provides important insights into the ecology of mosquito RNA viruses.Studies of virus ecology have generally focused on individual viral species. However, recent advances in bulk RNA sequencing make it possible to utilize metatranscriptomic approaches to reveal both complete virus diversity and the relative abundance of these viruses. We used such a metatranscriptomic approach to determine key aspects of the ecology of mosquito viruses in Western Australia. Our results show that RNA viruses are some of the most important components of the mosquito transcriptome, and we identified 19 new virus species from a diverse set of virus families. A key result was that host genetic background plays a more important role in shaping virus diversity than sampling location, withCulexspecies harboring more viruses at higher abundance than those fromAedesmosquitoes

**DOI:** [10.1128/JVI.00680-17](https://doi.org/10.1128/JVI.00680-17)
**URL:** https://doi.org/10.1128/JVI.00680-17
**Topics:** Mosquito
</details>


### Hot-Springs <a id='paper-hot-springs'></a>
<details>
<summary>**Identification of novel positive-strand RNA viruses by metagenomic analysis of archaea-dominated Yellowstone hot springs**</summary>

There are no known RNA viruses that infectArchaea. Filling this gap in our knowledge of viruses will enhance our understanding of the relationships between RNA viruses from the three domains of cellular life and, in particular, could shed light on the origin of the enormous diversity of RNA viruses infecting eukaryotes. We describe here the identification of novel RNA viral genome segments from high-temperature acidic hot springs in Yellowstone National Park in the United States. These hot springs harbor low-complexity cellular communities dominated by several species of hyperthermophilicArchaea. A viral metagenomics approach was taken to assemble segments of these RNA virus genomes from viral populations isolated directly from hot spring samples. Analysis of these RNA metagenomes demonstrated unique gene content that is not generally related to known RNA viruses ofBacteriaandEukarya. However, genes for RNA-dependent RNA polymerase (RdRp), a hallmark of positive-strand RNA viruses, were identified in two contigs. One of these contigs is approximately 5,600 nucleotides in length and encodes a polyprotein that also contains a region homologous to the capsid protein of nodaviruses, tetraviruses, and birnaviruses. Phylogenetic analyses of the RdRps encoded in these contigs indicate that the putative archaeal viruses form a unique group that is distinct from the RdRps of RNA viruses ofEukaryaandBacteria. Collectively, our findings suggest the existence of novel positive-strand RNA viruses that probably replicate in hyperthermophilic archaeal hosts and are highly divergent from RNA viruses that infect eukaryotes and even more distant from known bacterial RNA viruses. These positive-strand RNA viruses might be direct ancestors of RNA viruses of eukaryotes

**DOI:** [10.1128/JVI.07196-11](https://doi.org/10.1128/JVI.07196-11)
**URL:** https://doi.org/10.1128/JVI.07196-11
**Topics:** Hot-springs
</details>


### Soil <a id='paper-soil'></a>
<details>
<summary>**RNA-viromics reveals diverse communities of soil RNA viruses with the potential to affect grassland ecosystems across multiple trophic levels**</summary>

The distribution and diversity of RNA viruses in soil ecosystems are largely unknown, despite their significant impact on public health, ecosystem functions, and food security. Here, we characterise soil RNA viral communities along an altitudinal productivity gradient of peat, managed grassland and coastal soils. We identified 3462 viral contigs in RNA viromes from purified virus-like-particles in five soil-types and assessed their spatial distribution, phylogenetic diversity and potential host ranges. Soil types exhibited minimal similarity in viral community composition, but with >10-fold more viral contigs shared between managed grassland soils when compared with peat or coastal soils. Phylogenetic analyses predicted soil RNA viral communities are formed from viruses of bacteria, plants, fungi, vertebrates and invertebrates, with only 12% of viral contigs belonging to the bacteria-infecting Leviviricetes class. 11% of viral contigs were found to be most closely related to members of the Ourmiavirus genus, suggesting that members of this clade of plant viruses may be far more widely distributed and diverse than previously thought. These results contrast with soil DNA viromes which are typically dominated by bacteriophages. RNA viral communities, therefore, have the potential to exert influence on inter-kingdom interactions across terrestrial biomes

**DOI:** [10.1038/s43705-022-00110-x](https://doi.org/10.1038/s43705-022-00110-x)
**URL:** https://doi.org/10.1038/s43705-022-00110-x
**Topics:** soil
</details>


<details>
<summary>**RNA viromes from terrestrial sites across China expand environmental viral diversity**</summary>

RNA viromes from terrestrial sites across China expand environmental viral diversity

**DOI:** [10.1038/s41564-022-01180-2](https://doi.org/10.1038/s41564-022-01180-2)
**URL:** https://doi.org/10.1038/s41564-022-01180-2
**Topics:** soil
</details>


<details>
<summary>**Metatranscriptomic reconstruction reveals RNA viruses with the potential to shape carbon cycling in soil**</summary>

The diversity and ecology of RNA viruses is severely understudied in complex environments. Here we investigate the diversity and community patterns of soil RNA viruses by analyzing assembled metatranscriptomes. We tracked RNA viral and host communities for 22 d in 2 soil environments central to carbon cycling, the rhizosphere and detritosphere. This work is an important step toward understanding the factors that drive RNA viral communities. The main hosts in our system may be Fungi and Proteobacteria; this is in contrast to the ocean, where diatoms and other single cellular eukaryotes are primary hosts for RNA viruses. These results greatly expand the known diversity of viruses and contribute to understanding microbial interactions in soil with major implications for carbon cycling

**DOI:** [10.1073/pnas.1908291116](https://doi.org/10.1073/pnas.1908291116)
**URL:** https://doi.org/10.1073/pnas.1908291116
**Topics:** soil
</details>


### Flavi <a id='paper-flavi'></a>
<details>
<summary>**Divergent Viruses Discovered in Arthropods and Vertebrates Revise the Evolutionary History of the Flaviviridae and Related Viruses**</summary>

Viruses of the family Flaviviridae are important pathogens of humans and other animals and are currently classified into four genera. To better understand their diversity, evolutionary history, and genomic flexibility, we used transcriptome sequencing (RNA-seq) to search for the viruses related to theFlaviviridae in a range of potential invertebrate and vertebrate hosts. Accordingly, we recovered the full genomes of five segmented jingmenviruses and 12 distant relatives of the knownFlaviviridae (‚Äúflavi-like‚Äù viruses) from a range of arthropod species. Although these viruses are highly divergent, they share a similar genomic plan and common ancestry with theFlaviviridae in the NS3 and NS5 regions. Remarkably, although these viruses fill in major gaps in the phylogenetic diversity of theFlaviviridae , genomic comparisons reveal important changes in genome structure, genome size, and replication/gene regulation strategy during evolutionary history. In addition, the wide diversity of flavi-like viruses found in invertebrates, as well as their deep phylogenetic positions, suggests that they may represent the ancestral forms from which the vertebrate-infecting viruses evolved. For the vertebrate viruses, we expanded the previously mammal-only pegivirus-hepacivirus group to include a virus from the graceful catshark (Proscyllium habereri ), which in turn implies that these viruses possess a larger host range than is currently known. In sum, our data show that theFlaviviridae infect a far wider range of hosts and exhibit greater diversity in genome structure than previously anticipatedIMPORTANCEThe familyFlaviviridae of RNA viruses contains several notorious human pathogens, including dengue virus, West Nile virus, and hepatitis C virus. To date, however, our understanding of the biodiversity and evolution of theFlaviviridae has largely been directed toward vertebrate hosts and their blood-feeding arthropod vectors. Therefore, we investigated an expanded group of potential arthropod and vertebrate host species that have generally been ignored by surveillance programs. Remarkably, these species contained diverse flaviviruses and related viruses that are characterized by major changes in genome size and genome structure, such that these traits are more flexible than previously thought. More generally, these data suggest that arthropods may be the ultimate reservoir of theFlaviviridae and related viruses, harboring considerable genetic and phenotypic diversity. In sum, this study revises the traditional view on the evolutionary history, host range, and genomic structures of a major group of RNA viruses

**DOI:** [10.1128/JVI.02036-15](https://doi.org/10.1128/JVI.02036-15)
**URL:** https://doi.org/10.1128/JVI.02036-15
**Topics:** flavi
</details>


<details>
<summary>**A ~40-kb flavi-like virus does not encode a known error-correcting mechanism**</summary>

It is commonly held that there is a fundamental relationship between genome size and error rate, manifest as a notional ‚Äúerror threshold‚Äù that sets an upper limit on genome sizes. The genome sizes of RNA viruses, which have intrinsically high mutation rates due to a lack of mechanisms for error correction, must therefore be small to avoid accumulating an excessive number of deleterious mutations that will ultimately lead to population extinction. The proposed exceptions to this evolutionary rule are RNA viruses from the orderNidovirales(such as coronaviruses) that encode error-correcting exonucleases, enabling them to reach genome lengths greater than 40 kb. The recent discovery of large-genome flavi-like viruses (Flaviviridae), which comprise genomes up to 27 kb in length yet seemingly do not encode exonuclease domains, has led to the proposal that a proofreading mechanism is required to facilitate the expansion of nonsegmented RNA virus genomes above 30 kb. Herein, we describe a ~40 kb flavi-like virus identified in aHaliclonasponge metatranscriptome that does not encode a known exonuclease. Structural analysis revealed that this virus may have instead captured cellular domains associated with nucleic acid metabolism that have not been previously found in RNA viruses. Phylogenetic inference placed this virus as a divergent pesti-like lineage, such that we have provisionally termed it ‚ÄúMaximus pesti-like virus.‚Äù This virus represents an instance of a flavi-like virus achieving a genome size comparable to that of theNidoviralesand demonstrates that RNA viruses have evolved multiple solutions to overcome the error threshold

**DOI:** [10.1073/pnas.2403805121](https://doi.org/10.1073/pnas.2403805121)
**URL:** https://doi.org/10.1073/pnas.2403805121
**Topics:** flavi
</details>


### Amphibians <a id='paper-amphibians'></a>
<details>
<summary>**Divergent Influenza-Like Viruses of Amphibians and Fish Support an Ancient Evolutionary Association**</summary>

Influenza viruses (family Orthomyxoviridae) infect a variety of vertebrates, including birds, humans, and other mammals. Recent metatranscriptomic studies have uncovered divergent influenza viruses in amphibians, fish and jawless vertebrates, suggesting that these viruses may be widely distributed. We sought to identify additional vertebrate influenza-like viruses through the analysis of publicly available RNA sequencing data. Accordingly, by data mining, we identified the complete coding segments of five divergent vertebrate influenza-like viruses. Three fell as sister lineages to influenza B virus: salamander influenza-like virus in Mexican walking fish (Ambystoma mexicanum) and plateau tiger salamander (Ambystoma velasci), Siamese algae-eater influenza-like virus in Siamese algae-eater fish (Gyrinocheilus aymonieri) and chum salmon influenza-like virus in chum salmon (Oncorhynchus keta). Similarly, we identified two influenza-like viruses of amphibians that fell as sister lineages to influenza D virus: cane toad influenza-like virus and the ornate chorus frog influenza-like virus, in the cane toad (Rhinella marina) and ornate chorus frog (Microhyla fissipes), respectively. Despite their divergent phylogenetic positions, these viruses retained segment conservation and splicing consistent with transcriptional regulation in influenza B and influenza D viruses, and were detected in respiratory tissues. These data suggest that influenza viruses have been associated with vertebrates for their entire evolutionary history

**DOI:** [10.3390/v12091042](https://doi.org/10.3390/v12091042)
**URL:** https://doi.org/10.3390/v12091042
**Topics:** Amphibians, fish
</details>


<details>
<summary>**Viral Discovery in the Invasive Australian Cane Toad (Rhinella marina) Using Metatranscriptomic and Genomic Approaches**</summary>

Cane toads are poisonous amphibians that were introduced to Australia in 1935 for insect control. Since then, their population has increased dramatically, and they now threaten many native Australian species. One potential method to control the population is to release a cane toad virus with high mortality rates, yet few cane toad viruses have been characterized. This study samples cane toads from different Australian locations and uses an RNA sequencing and computational approach to find new viruses. We report novel complete picornavirus and retrovirus sequences that were genetically similar to viruses infecting frogs, reptiles, and fish. Using data generated in other studies, we show that these viral sequences are present in cane toads from distinct Australian locations. Three sequences related to circoviruses were also found in the toad genome. The identification of new viral sequences will aid future studies that investigate their prevalence and potential as agents for biocontrol

**DOI:** [10.1128/JVI.00768-18](https://doi.org/10.1128/JVI.00768-18)
**URL:** https://doi.org/10.1128/JVI.00768-18
**Topics:** Amphibians
</details>


### Mosquitoes <a id='paper-mosquitoes'></a>
<details>
<summary>**Negeviruses found in multiple species of mosquitoes from southern Portugal: Isolation, genetic diversity, and replication in insect cell cultur**</summary>

Negeviruses found in multiple species of mosquitoes from southern Portugal: Isolation, genetic diversity, and replication in insect cell culture

**DOI:** [10.1016/j.virol.2015.04.021](https://doi.org/10.1016/j.virol.2015.04.021)
**URL:** https://doi.org/10.1016/j.virol.2015.04.021
**Topics:** mosquitoes
</details>


<details>
<summary>**RNA shotgun metagenomic sequencing of northern California (USA) mosquitoes uncovers viruses, bacteria, and fungi**</summary>

RNA shotgun metagenomic sequencing of northern California (USA) mosquitoes uncovers viruses, bacteria, and fungi

**DOI:** [10.3389/fmicb.2015.00185](https://doi.org/10.3389/fmicb.2015.00185)
**URL:** https://doi.org/10.3389/fmicb.2015.00185
**Topics:** mosquitoes
</details>


<details>
<summary>**Meta-Transcriptomic Comparison of the RNA Viromes of the Mosquito Vectors Culex pipiens and Culex torrentium in Northern Europe**</summary>

Mosquitoes harbor an extensive diversity of ‚Äòinsect-specific‚Äô RNA viruses in addition to those important to human and animal health. However, because most studies of the mosquito virome have been conducted at lower latitudes, little is known about the diversity and evolutionary history of RNA viruses sampled from mosquitoes in northerly regions. Here, we compared the RNA virome of two common northern mosquito species, Culex pipiens and Culex torrentium, collected in south-central Sweden. Following bulk RNA-sequencing (meta-transcriptomics) of 12 libraries, comprising 120 specimens of Cx. pipiens and 150 specimens of Cx. torrentium, we identified 40 viruses (representing 14 virus families) of which 28 were novel based on phylogenetic analysis of the RNA-dependent RNA polymerase (RdRp) protein. Hence, we documented similar levels of virome diversity as in mosquitoes sampled from the more biodiverse lower latitudes. Many viruses were also related to those sampled on other continents, indicative of a widespread global movement and/or long host‚Äìvirus co-evolution. Although the two mosquito species investigated have overlapping geographical distributions and share many viruses, several viruses were only found at a specific location at this scale of sampling, such that local habitat and geography may play an important role in shaping viral diversity in Culex mosquitoes

**DOI:** [10.3390/v11111033](https://doi.org/10.3390/v11111033)
**URL:** https://doi.org/10.3390/v11111033
**Topics:** mosquitoes
</details>


### Surveillance <a id='paper-surveillance'></a>
<details>
<summary>**FLDS: A Comprehensive dsRNA Sequencing Method for Intracellular RNA Virus Surveillance**</summary>

FLDS: A Comprehensive dsRNA Sequencing Method for Intracellular RNA Virus Surveillance

**DOI:** [10.1264/jsme2.ME15171](https://doi.org/10.1264/jsme2.ME15171)
**URL:** https://doi.org/10.1264/jsme2.ME15171
**Topics:** surveillance, methodology
</details>


<details>
<summary>**Enhanced arbovirus surveillance with deep sequencing: Identification of novel rhabdoviruses and bunyaviruses in Australian mosquitoes**</summary>

Enhanced arbovirus surveillance with deep sequencing: Identification of novel rhabdoviruses and bunyaviruses in Australian mosquitoes

**DOI:** [10.1016/j.virol.2013.09.026](https://doi.org/10.1016/j.virol.2013.09.026)
**URL:** https://doi.org/10.1016/j.virol.2013.09.026
**Topics:** surveillance, Mosquito
</details>


### -Ssrna <a id='paper--ssrna'></a>
<details>
<summary>**Unprecedented genomic diversity of RNA viruses in arthropods reveals the ancestry of negative-sense RNA viruses**</summary>

Although arthropods are important viral vectors, the biodiversity of arthropod viruses, as well as the role that arthropods have played in viral origins and evolution, is unclear. Through RNA sequencing of 70 arthropod species we discovered 112 novel viruses that appear to be ancestral to much of the documented genetic diversity of negative-sense RNA viruses, a number of which are also present as endogenous genomic copies. With this greatly enriched diversity we revealed that arthropods contain viruses that fall basal to major virus groups, including the vertebrate-specific arenaviruses, filoviruses, hantaviruses, influenza viruses, lyssaviruses, and paramyxoviruses. We similarly documented a remarkable diversity of genome structures in arthropod viruses, including a putative circular form, that sheds new light on the evolution of genome organization. Hence, arthropods are a major reservoir of viral genetic diversity and have likely been central to viral evolution

**DOI:** [10.7554/eLife.05378](https://doi.org/10.7554/eLife.05378)
**URL:** https://doi.org/10.7554/eLife.05378
**Topics:** -ssRNA, Arthropods
</details>


### Hexapods <a id='paper-hexapods'></a>
<details>
<summary>**Analysis of the RNA virome of basal hexapods**</summary>

The diversity and evolution of RNA viruses has been well studied in arthropods and especially in insects. However, the diversity of RNA viruses in the basal hexapods has not been analysed yet. To better understand their diversity, evolutionary histories and genome organizations, we searched for RNA viruses in transcriptome and genome databases of basal hexapods. We discovered ¬†40 novel RNA viruses, some of which are also present as endogenous viral elements derived from RNA viruses. Here, we demonstrated that basal hexapods host 14 RNA viral clades that have been recently identified in invertebrates. The following RNA viral clades are associated with basal hexapods: Reo, Partiti-Picobirna, Toti-Chryso, Mono-Chu, Bunya-Arena, Orthomyxo, Qinvirus, Picorna-Calici, Hepe-Virga, Narna-Levi, Tombus-Noda, Luteo-Sobemo, Permutotetra and Flavi. We have found representatives of the nine RNA viral clades that are present as endogenous genomic copies in the genomes of Machilis (Monocondylia) and Catajapyx (Diplura). Our study provided a first insight into the diversity of RNA viruses in basal hexapods and demonstrated that the basal hexapods possess quite high diversity of RNA viral clades

**DOI:** [10.7717/peerj.8336](https://doi.org/10.7717/peerj.8336)
**URL:** https://doi.org/10.7717/peerj.8336
**Topics:** hexapods
</details>


### Ecology <a id='paper-ecology'></a>
<details>
<summary>**Human land use impacts viral diversity and abundance in a New Zealand river**</summary>

Although water-borne viruses have important implications for the health of humans and other animals, little is known about the impact of human land use on viral diversity and evolution in water systems such as rivers. We used metatranscriptomic sequencing to compare the diversity and abundance of viruses at sampling sites along a single river in New Zealand that differed in human land-use impacts, ranging from pristine to urban. From this, we identified 504 putative virus species, of which 97‚Äâper cent were novel. Many of the novel viruses were highly divergent and likely included a new subfamily within the Parvoviridae. We identified at least sixty-three virus species that may infect vertebrates‚Äîmost likely fish and water birds‚Äîfrom the Astroviridae, Birnaviridae, Parvoviridae, and Picornaviridae. No putative human viruses were detected. Importantly, we observed differences in the composition of viral communities at sites impacted by human land use (farming and urban) compared to native forest sites (pristine). At the viral species level, the urban sites had higher diversity (327 virus species) than the farming (n‚Äâ=‚Äâ150) and pristine sites (n‚Äâ=‚Äâ119), and more viruses were shared between the urban and farming sites (n‚Äâ=‚Äâ76) than between the pristine and farming or urban sites (n‚Äâ=‚Äâ24). The two farming sites had a lower viral abundance across all host types, while the pristine sites had a higher abundance of viruses associated with animals, plants, and fungi. We also identified viruses linked to agriculture and human impact at the river sampling sites in farming and urban areas that were not present at the native forest sites. Although based on a small sample size, our study suggests that human land use can impact viral communities in rivers, such that further work is needed to reduce the impact of intensive farming and urbanisation on water systems

**DOI:** [10.1093/ve/veac032](https://doi.org/10.1093/ve/veac032)
**URL:** https://doi.org/10.1093/ve/veac032
**Topics:** ecology
</details>


<details>
<summary>**Host phylogeny shapes viral transmission networks in an island ecosystem**</summary>

Virus transmission between host species underpins disease emergence. Both host phylogenetic relatedness and aspects of their ecology, such as species interactions and predator‚Äìprey relationships, may govern rates and patterns of cross-species virus transmission and hence zoonotic risk. To address the impact of host phylogeny and ecology on virus diversity and evolution, we characterized the virome structure of a relatively isolated island ecological community in Fiordland, New Zealand, that are linked through a food web. We show that phylogenetic barriers that inhibited cross-species virus transmission occurred at the level of host phyla (between the Chordata, Arthropoda and Streptophyta) as well as at lower taxonomic levels. By contrast, host ecology, manifest as predator‚Äìprey interactions and diet, had a smaller influence on virome composition, especially at higher taxonomic levels. The virus‚Äìhost community comprised a ‚Äòsmall world‚Äô network, in which hosts with a high diversity of viruses were more likely to acquire new viruses, and generalist viruses that infect multiple hosts were more likely to infect additional species compared to host specialist viruses. Such a highly connected ecological community increases the likelihood of cross-species virus transmission, particularly among closely related species, and suggests that host generalist viruses present the greatest risk of disease emergence

**DOI:** [10.1038/s41559-023-02192-9](https://doi.org/10.1038/s41559-023-02192-9)
**URL:** https://www.nature.com/articles/s41559-023-02192-9
**Topics:** ecology, phylogeny
</details>


<details>
<summary>**Abundant and Diverse RNA Viruses in Insects Revealed by RNA-Seq Analysis: Ecological and Evolutionary Implications**</summary>

Insects comprise the largest proportion of animals on earth and are frequently implicated in the transmission of vector-borne diseases. However, considerable attention has been paid to the phytophagous and hematophagous insects, with results that provide insufficient and biased information about the viruses in insects. Here, we have delivered compelling evidence for the exceptional abundance and genetic diversity of RNA viruses in a wide range of insects. Novel viruses were found to cover major categories of RNA viruses, and many formed novel clusters divergent from the previously described taxa, dramatically broadening the range of known RNA viruses in insects. These newly characterized RNA viruses exhibited high levels of genomic plasticity in genome size, open reading frame (ORF) number, intergenic structure, and gene rearrangement and segmentation. This work provides comprehensive insight into the origin, spread, and evolution of RNA viruses. Of course, a large-scale virome project involving more organisms would provide more-detailed information about the virus infections in insects

**DOI:** [10.1128/mSystems.00039-20](https://doi.org/10.1128/mSystems.00039-20)
**URL:** https://doi.org/10.1128/mSystems.00039-20
**Topics:** Ecology, insect
</details>


### Plants <a id='paper-plants'></a>
<details>
<summary>**Transcriptome Mining Expands Knowledge of RNA Viruses across the Plant Kingdom**</summary>

Our knowledge of plant viruses is mainly limited to those infecting economically important host species. In particular, we know little about those viruses infecting basal plant lineages such as the ferns, lycophytes, bryophytes, and charophytes

**DOI:** [10.1128/jvi.00260-22](https://doi.org/10.1128/jvi.00260-22)
**URL:** https://doi.org/10.1128/jvi.00260-22
**Topics:** plants
</details>


<details>
<summary>**A Survey Using High-Throughput Sequencing Suggests That the Diversity of Cereal and Barley Yellow Dwarf Viruses Is Underestimated**</summary>

Worldwide, barley/cereal yellow dwarf viruses (YDVs) are the most widespread and damaging group of cereal viruses. In this study, we applied high-throughput sequencing technologies (HTS) to perform a virus survey on symptomatic plants from 47 cereal fields in Estonia. HTS allowed the assembly of complete genome sequences for 22 isolates of cereal yellow dwarf virus RPS, barley yellow dwarf virus GAV, barley yellow dwarf virus PAS (BYDV-PAS), barley yellow dwarf virus PAV (BYDV-PAV), and barley yellow dwarf virus OYV (BYDV-OYV). We also assembled a near-complete genome of the putative novel species BYDV-OYV from Swedish samples of meadow fescue. Previously, partial sequencing of the central part of the coat protein gene indicated that BYDV-OYV represented a putative new species closely related to BYDV-PAV-CN, which currently is recognized as a subtype of BYDV-PAV. The present study found that whereas the 3‚Ä≤gene block of BYDV-OYV shares the closest relationship with BYDV-PAV-CN, the 5‚Ä≤gene block of BYDV-OYV shows the closest relationships to that of BYDV-PAS. Recombination detection analysis revealed that BYDV-OYV is a parental virus for both. Analysis of complete genome sequence data indicates that both BYDV-OYV and BYDV-PAV-CN meet the species criteria of genus Luteovirus. The study discusses BYDV phylogeny, and through a systematic in silico analysis of published primers for YDV detection, the existing gaps in current diagnostic practices for detection of YDVs, proposing primer pairs based on the most recent genomic information for the detection of different BYDV species. Thanks to the rising number of sequences available in databases, continuous updating of diagnostic primers can improve test specificity, e.g., inclusivity and exclusivity at species levels. This is needed to properly survey the geographical and host distribution of the different species of the YDV complex and their prevalence in cereal/barley yellow dwarf disease epidemics

**DOI:** [10.3389/fmicb.2021.673218](https://doi.org/10.3389/fmicb.2021.673218)
**URL:** https://doi.org/10.3389/fmicb.2021.673218
**Topics:** plants
</details>


<details>
<summary>**Metagenomic identification of novel viruses of maize and teosinte in North America**</summary>

Maize-infecting viruses are known to inflict significant agronomic yield loss throughout the world annually. Identification of known or novel causal agents of disease prior to outbreak is imperative to preserve food security via future crop protection efforts. Toward this goal, a large-scale metagenomic approach utilizing high throughput sequencing (HTS) was employed to identify novel viruses with the potential to contribute to yield loss of graminaceous species, particularly maize, in North America            Here we present four novel viruses discovered by HTS and individually validated by Sanger sequencing. Three of these viruses are RNA viruses belonging to either the Betaflexiviridae or Tombusviridae families. Additionally, a novel DNA virus belonging to the Geminiviridae family was discovered, the first Mastrevirus identified in North American maize        Conclusions    Metagenomic studies of crop and crop-related species such as this may be useful for the identification and surveillance of known and novel viral pathogens of crops. Monitoring related species may prove useful in identifying viruses capable of infecting crops due to overlapping insect vectors and viral host-range to protect food security

**DOI:** [10.1186/s12864-022-09001-w](https://doi.org/10.1186/s12864-022-09001-w)
**URL:** https://doi.org/10.1186/s12864-022-09001-w
**Topics:** plants
</details>


<details>
<summary>**Soybean Thrips (Thysanoptera: Thripidae) Harbor Highly Diverse Populations of Arthropod, Fungal and Plant Viruses**</summary>

Soybean thrips (Neohydatothrips variabilis) are one of the most efficient vectors of soybean vein necrosis virus, which can cause severe necrotic symptoms in sensitive soybean plants. To determine which other viruses are associated with soybean thrips, the metatranscriptome of soybean thrips, collected by the Midwest Suction Trap Network during 2018, was analyzed. Contigs assembled from the data revealed a remarkable diversity of virus-like sequences. Of the 181 virus-like sequences identified, 155 were novel and associated primarily with taxa of arthropod-infecting viruses, but sequences similar to plant and fungus-infecting viruses were also identified. The novel viruses were predicted to have positive-sense RNA, negative-stranded RNA, double-stranded RNA, and single-stranded DNA genomes. The assembled sequences included 100 contigs that represented at least 95% coverage of a virus genome or genome segment. Sequences represented 12 previously described arthropod viruses including eight viruses reported from Hubei Province in China, and 12 plant virus sequences of which six have been previously described. The presence of diverse populations of plant viruses within soybean thrips suggests they feed on and acquire viruses from multiple host plant species that could be transmitted to soybean. Assessment of the virome of soybean thrips provides, for the first time, information on the diversity of viruses present in thrips

**DOI:** [10.3390/v12121376](https://doi.org/10.3390/v12121376)
**URL:** https://doi.org/10.3390/v12121376
**Topics:** plants
</details>


### Viroids <a id='paper-viroids'></a>
<details>
<summary>**Viroid-like RNA-dependent RNA polymerase-encoding ambiviruses are abundant in complex fungi**</summary>

Ambiviruses are hybrid infectious elements encoding the hallmark gene of RNA viruses, the RNA-dependent RNA polymerase, and self-cleaving RNA ribozymes found in many viroids. Ambiviruses are thought to be pathogens of fungi, although the majority of reported genomes have been identified in metatranscriptomes. Here, we present a comprehensive screen for ambiviruses in more than 46,500 fungal transcriptomes from the Sequence Read Archive (SRA). Our data-driven virus discovery approach identified more than 2,500 ambiviral sequences across the kingdom Fungi with a striking expansion in members of the phylum Basidiomycota representing the most complex fungal organisms. Our study unveils a large diversity of unknown ambiviruses with as little as 27% protein sequence identity to known members and sheds new light on the evolution of this distinct class of infectious agents with RNA genomes. No evidence for the presence of ambiviruses in human microbiomes was obtained from a comprehensive screen of respective metatranscriptomes available in the SRA

**DOI:** [10.3389/fmicb.2023.1144003](https://doi.org/10.3389/fmicb.2023.1144003)
**URL:** https://doi.org/10.3389/fmicb.2023.1144003
**Topics:** viroids, fungi, ambiviruses
</details>


### Animalia <a id='paper-animalia'></a>
<details>
<summary>**A new lineage of segmented RNA viruses infecting animals**</summary>

Metagenomic sequencing has revolutionised our knowledge of virus diversity, with new virus sequences being reported faster than ever before. However, virus discovery from metagenomic sequencing usually depends on detectable homology: without a sufficiently close relative, so-called ‚Äòdark‚Äô virus sequences remain unrecognisable. An alternative approach is to use virus-identification methods that do not depend on detecting homology, such as virus recognition by host antiviral immunity. For example, virus-derived small RNAs have previously been used to propose ‚Äòdark‚Äô virus sequences associated with the Drosophilidae (Diptera). Here, we combine published Drosophila data with a comprehensive search of transcriptomic sequences and selected meta-transcriptomic datasets to identify a completely new lineage of segmented positive-sense single-stranded RNA viruses that we provisionally refer to as the Quenyaviruses. Each of the five segments contains a single open reading frame, with most encoding proteins showing no detectable similarity to characterised viruses, and one sharing a small number of residues with the RNA-dependent RNA polymerases of single- and double-stranded RNA viruses. Using these sequences, we identify close relatives in approximately 20 arthropods, including insects, crustaceans, spiders, and a myriapod. Using a more conserved sequence from the putative polymerase, we further identify relatives in meta-transcriptomic datasets from gut, gill, and lung tissues of vertebrates, reflecting infections of vertebrates or of their associated parasites. Our data illustrate the utility of small RNAs to detect viruses with limited sequence conservation, and provide robust evidence for a new deeply divergent and phylogenetically distinct RNA virus lineage

**DOI:** [10.1093/ve/vez061](https://doi.org/10.1093/ve/vez061)
**URL:** https://doi.org/10.1093/ve/vez061
**Topics:** Animalia
</details>


<details>
<summary>**Rapid identification of known and new RNA viruses from animal tissues**</summary>

Rapid Identification of Known and New RNA Viruses from Animal Tissues

**DOI:** [10.1371/journal.ppat.1000163](https://doi.org/10.1371/journal.ppat.1000163)
**URL:** https://doi.org/10.1371/journal.ppat.1000163
**Topics:** Animalia
</details>


<details>
<summary>**Diverse RNA viruses in the venom-related microenvironment of different animal phyla**</summary>

Venom is known as the source of natural antimicrobial products. Previous studies have largely focused on the expression of venom-related genes and the biochemical components of venom. With the advent of metagenomic sequencing, many more microorganisms, especially viruses, have been identified in highly diverse environments. Herein, we investigated the RNA virome in the venom-related microenvironment through analysis of a large volume of venom-related RNA-sequencing data mined from public databases. From this, we identified viral sequences belonging to thirty-six different viruses, of which twenty-two were classified as ‚Äònovel‚Äô as they exhibited less than 90‚Äâper cent amino acid identity to known viruses in the RNA-dependent RNA polymerase. Most of these novel viruses possessed genome structures similar to their closest relatives, with specific alterations in some cases. Phylogenetic analyses revealed that these viruses belonged to at least twenty-two viral families or unclassified groups, some of which were highly divergent from known taxa. Although further analysis failed to find venom-specific viruses, some viruses seemingly had much higher abundance in the venom-related microenvironment than in other tissues. In sum, our study provides insights into the RNA virome of the venom-related microenvironment from diverse animal phyla

**DOI:** [10.1093/ve/veae024](https://doi.org/10.1093/ve/veae024)
**URL:** https://academic.oup.com/ve/article/10/1/veae024/7619247?login=false
**Topics:** Animalia
</details>


### Invertebrate <a id='paper-invertebrate'></a>
<details>
<summary>**Redefining the invertebrate RNA virosphere**</summary>

Redefining the invertebrate RNA virosphere

**DOI:** [10.1038/nature20167](https://doi.org/10.1038/nature20167)
**URL:** https://doi.org/10.1038/nature20167
**Topics:** invertebrate
</details>


### Insect <a id='paper-insect'></a>
<details>
<summary>**Composition and global distribution of the mosquito virome - A comprehensive database of insect-specific viruses**</summary>

Composition and global distribution of the mosquito virome - A comprehensive database of insect-specific viruses

**DOI:** [10.1016/j.onehlt.2023.100490](https://doi.org/10.1016/j.onehlt.2023.100490)
**URL:** https://doi.org/10.1016/j.onehlt.2023.100490
**Topics:** insect
</details>


<details>
<summary>**Identification of diverse viruses associated with grasshoppers unveils the parallel relationship between host phylogeny and virome composition**</summary>

Grasshoppers (Orthoptera: Acridoidea) are one of the most dangerous agricultural pests. Environmentally benign microbial pesticides are increasingly desirable for controlling grasshopper outbreaks in fragile ecosystems. However, little is known about natural pathogens infecting this pest. Here we profile the rich viral communities in forty-five grasshopper species and report 302 viruses, including 231 novel species. Most of the identified viruses are related to other insect viruses, and small RNA sequencing indicates that some are targeted by host antiviral RNA interference (RNAi) pathway. Our analysis of relationships between host phylogeny and virus diversity suggests that the composition of viromes is closely allied with host evolution. Overall, this study is a first extensive exploration of viruses in grasshoppers and provides a valuable comparative dataset of both academic and applied interest

**DOI:** [10.1093/ve/veac057](https://doi.org/10.1093/ve/veac057)
**URL:** https://doi.org/10.1093/ve/veac057
**Topics:** insect, grasshopper
</details>


<details>
<summary>**A Diverse Range of Novel RNA Viruses in Geographically Distinct Honey Bee Populations**</summary>

Understanding the diversity and consequences of viruses present in honey bees is critical for maintaining pollinator health and managing the spread of disease. The viral landscape of honey bees (Apis mellifera) has changed dramatically since the emergence of the parasitic miteVarroa destructor, which increased the spread of virulent variants of viruses such as deformed wing virus. Previous genomic studies have focused on colonies suffering from infections by Varroaand virulent viruses, which could mask other viral species present in honey bees, resulting in a distorted view of viral diversity. To capture the viral diversity within colonies that are exposed to mites but do not suffer the ultimate consequences of the infestation, we examined populations of honey bees that have evolved naturally or have been selected for resistance toVarroa. This analysis revealed seven novel viruses isolated from honey bees sampled globally, including the first identification of negative-sense RNA viruses in honey bees. Notably, two rhabdoviruses were present in three geographically diverse locations and were also present inVarroamites parasitizing the bees. To characterize the antiviral response, we performed deep sequencing of small RNA populations in honey bees and mites. This provided evidence of a Dicer-mediated immune response in honey bees, while the viral small RNA profile inVarroamites was novel and distinct from the response observed in bees. Overall, we show that viral diversity in honey bee colonies is greater than previously thought, which encourages additional studies of the bee virome on a global scale and which may ultimately improve disease management.Honey bee populations have become increasingly susceptible to colony losses due to pathogenic viruses spread by parasiticVarroamites. To date, 24 viruses have been described in honey bees, with most belonging to the orderPicornavirales. CollapsingVarroa-infected colonies are often overwhelmed with high levels of picornaviruses. To examine the underlying viral diversity in honey bees, we employed viral metatranscriptomics analyses on three geographically diverseVarroa-resistant populations from Europe, Africa, and the Pacific. We describe seven novel viruses from a range of diverse viral families, including two viruses that are present in all three locations. In honey bees, small RNA sequences indicate that these viruses are processed by Dicer and the RNA interference pathway, whereasVarroamites produce strikingly novel small RNA patterns. This work increases the number and diversity of known honey bee viruses and will ultimately contribute to improved disease management in our most important agricultural pollinator

**DOI:** [10.1128/JVI.00158-17](https://doi.org/10.1128/JVI.00158-17)
**URL:** https://doi.org/10.1128/JVI.00158-17
**Topics:** insect
</details>


<details>
<summary>**The diversity and evolutionary relationships of ticks and tick-borne bacteria collected in China**</summary>

Ticks (order Ixodida) are ectoparasites, vectors and reservoirs of many infectious agents affecting humans and domestic animals. However, the lack of information on tick genomic diversity leaves significant gaps in the understanding of the evolution of ticks and associated bacteria            We collected > 20,000 contemporary and historical (up to 60¬†years of preservation) tick samples representing a wide range of tick biodiversity across diverse geographic regions in China. Metagenomic sequencing was performed on individual ticks to obtain the complete or near-complete mitochondrial (mt) genome sequences from 46 tick species, among which mitochondrial genomes of 23 species were recovered for the first time. These new mt genomes data greatly expanded the diversity of many tick groups and revealed five cryptic species. Utilizing the same metagenomic sequence data we identified divergent and abundant bacteria in Haemaphysalis, Ixodes, Dermacentor and Carios ticks, including nine species of pathogenetic bacteria and potentially new species within the genus Borrelia. We also used these data to explore the evolutionary relationship between ticks and their associated bacteria, revealing a pattern of long-term co-divergence relationship between ticks and Rickettsia and Coxiella bacteria        Conclusions    In sum, our study provides important new information on the genetic diversity of ticks based on an analysis of mitochondrial DNA as well as on the prevalence of tick-borne pathogens in China. It also sheds new light on the long-term evolutionary and ecological relationships between ticks and their associated bacteria        Graphical Abstract

**DOI:** [10.1186/s13071-022-05485-3](https://doi.org/10.1186/s13071-022-05485-3)
**URL:** https://doi.org/10.1186/s13071-022-05485-3
**Topics:** insect
</details>


<details>
<summary>**Twenty-Five New Viruses Associated with the Drosophilidae (Diptera)**</summary>

Drosophila melanogaster is an important laboratory model for studies of antiviral immunity in invertebrates, and Drosophila species provide a valuable system to study virus host range and host switching. Here, we use metagenomic RNA sequencing of about 1600 adult flies to discover 25 new RNA viruses associated with six different drosophilid hosts in the wild. We also provide a comprehensive listing of viruses previously reported from the Drosophilidae. The new viruses include Iflaviruses, Rhabdoviruses, Nodaviruses, and Reoviruses, and members of unclassified lineages distantly related to Negeviruses, Sobemoviruses, Poleroviruses, Flaviviridae, and Tombusviridae. Among these are close relatives of Drosophila X virus and Flock House virus, which we find in association with wild Drosophila immigrans. These two viruses are widely used in experimental studies but have not been previously reported to naturally infect Drosophila. Although we detect no new DNA viruses, in D. immigrans and Drosophila obscura, we identify sequences very closely related to Armadillidium vulgare iridescent virus (Invertebrate iridescent virus 31), bringing the total number of DNA viruses found in the Drosophilidae to three.

**DOI:** [10.4137/EBO.S39454](https://doi.org/10.4137/EBO.S39454)
**URL:** https://doi.org/10.4137/EBO.S39454
**Topics:** insect
</details>


<details>
<summary>**The virome of Drosophila suzukii, an invasive pest of soft fruit**</summary>

The virome of Drosophila suzukii, an invasive pest of soft fruit

**DOI:** [10.1093/ve/vey009](https://doi.org/10.1093/ve/vey009)
**URL:** https://doi.org/10.1093/ve/vey009
**Topics:** insect, Drosophila
</details>


<details>
<summary>**The Distribution, and Evolution of Viruses Associated with Drosophila melanogaster**</summary>

The Discovery, Distribution, and Evolution of Viruses Associated with Drosophila melanogaster

**DOI:** [10.1371/journal.pbio.1002210](https://doi.org/10.1371/journal.pbio.1002210)
**URL:** https://doi.org/10.1371/journal.pbio.1002210
**Topics:** insect
</details>


<details>
<summary>**A Diverse Viral Community from Predatory Wasps in Their Native and Invaded Range, with a New Virus Infectious to Honey Bees**</summary>

Wasps of the genus Vespula are social insects that have become major pests and predators in their introduced range. Viruses present in these wasps have been studied in the context of spillover from honey bees, yet we lack an understanding of the endogenous virome of wasps as potential reservoirs of novel emerging infectious diseases. We describe the characterization of 68 novel and nine previously identified virus sequences found in transcriptomes of Vespula vulgaris in colonies sampled from their native range (Belgium) and an invasive range (New Zealand). Many viruses present in the samples were from the Picorna-like virus family (38%). We identified one Luteo-like virus, Vespula vulgaris Luteo-like virus 1, present in the three life stages examined in all colonies from both locations, suggesting this virus is a highly prevalent and persistent infection in wasp colonies. Additionally, we identified a novel Iflavirus with similarity to a recently identified Moku virus, a known wasp and honey bee pathogen. Experimental infection of honey bees with this novel Vespula vulgaris Moku-like virus resulted in an active infection. The high viral diversity present in these invasive wasps is a likely indication that their polyphagous diet is a rich source of viral infections

**DOI:** [10.3390/v13081431](https://doi.org/10.3390/v13081431)
**URL:** https://doi.org/10.3390/v13081431
**Topics:** insect
</details>


### Human <a id='paper-human'></a>
<details>
<summary>**A catalog of tens of thousands of viruses from human metagenomes reveals hidden associations with chronic diseases**</summary>

Mechanisms of many human chronic diseases involve abnormal action of the immune system and/or altered metabolism. The microbiome, an important regulator of metabolic and immune-related phenotypes, has been shown to be associated with or participate in the development of a variety of chronic diseases. Viruses of bacteria (i.e., ‚Äúphages‚Äù) are ubiquitous and mysterious, and several studies have shown that phages exert great control over the behavior‚Äîand misbehavior‚Äîof their host bacteria. This study uses techniques to discover and analyze over 45,000 viruses associated with human bodies. The abundance of over 2,000 specific phages is found to correlate with a variety of common chronic diseases

**DOI:** [10.1073/pnas.2023202118](https://doi.org/10.1073/pnas.2023202118)
**URL:** https://doi.org/10.1073/pnas.2023202118
**Topics:** human
</details>


### Bats <a id='paper-bats'></a>
<details>
<summary>**Metagenomic Analysis of Viruses from Bat Fecal Samples Reveals Many Novel Viruses in Insectivorous Bats in China**</summary>

Increasing data indicate that bats harbor diverse viruses, some of which cause severe human diseases. In this study, sequence-independent amplification and high-throughput sequencing (Solexa) were applied to the metagenomic analysis of viruses in bat fecal samples collected from 6 locations in China. A total of 8,746,417 reads with a length of 306,124,595 bp were obtained. Among these reads, 13,541 (0.15%) had similarity to phage sequences and 9,170 (0.1%) had similarity to eukaryotic virus sequences. A total of 129 assembled contigs (>100 nucleotides) were constructed and compared with GenBank: 32 contigs were related to phages, and 97 were related to eukaryotic viruses. The most frequent reads and contigs related to eukaryotic viruses were homologous to densoviruses, dicistroviruses, coronaviruses, parvoviruses, and tobamoviruses, a range that includes viruses from invertebrates, vertebrates, and plants. Most of the contigs had low identities to known viral genomic or protein sequences, suggesting that a large number of novel and genetically diverse insect viruses as well as putative mammalian viruses are transmitted by bats in China. This study provides the first preliminary understanding of the virome of some bat populations in China, which may guide the discovery and isolation of novel viruses in the future

**DOI:** [10.1128/JVI.06671-11](https://doi.org/10.1128/JVI.06671-11)
**URL:** https://doi.org/10.1128/JVI.06671-11
**Topics:** bats
</details>


<details>
<summary>**Individual bat virome analysis reveals co-infection and spillover among bats and virus zoonotic potential**</summary>

Bats are reservoir hosts for many zoonotic viruses. Despite this, relatively little is known about the diversity and abundance of viruses within individual bats, and hence the frequency of virus co-infection and spillover among them. We characterize the mammal-associated viruses in 149 individual bats sampled from Yunnan province, China, using an unbiased meta-transcriptomics approach. This reveals a high frequency of virus co-infection (simultaneous infection of bat individuals by multiple viral species) and spillover among the animals studied, which may in turn facilitate virus recombination and reassortment. Of note, we identify five viral species that are likely to be pathogenic to humans or livestock, based on phylogenetic relatedness to known pathogens or in vitro receptor binding assays. This includes a novel recombinant SARS-like coronavirus that is closely related to both SARS-CoV and SARS-CoV-2. In vitro assays indicate that this recombinant virus can utilize the human ACE2 receptor such that it is likely to be of increased emergence risk. Our study highlights the common occurrence of co-infection and spillover of bat viruses and their implications for virus emergence

**DOI:** [10.1038/s41467-023-39835-1](https://doi.org/10.1038/s41467-023-39835-1)
**URL:** https://www.nature.com/articles/s41467-023-39835-1
**Topics:** bats, zoonosis
</details>


<details>
<summary>**Highly diverse ribonucleic acid viruses in the viromes of eukaryotic host species in Yunnan province, China**</summary>

The diversity in currently documented viruses and their morphological characteristics indicates the need for understanding the evolutionary characteristics of viruses. Notably, further studies are needed to obtain a comprehensive landscape of virome, the virome of host species in Yunnan province, China  Materials and methodsWe implemented the metagenomic next-generation sequencing strategy to investigate the viral diversity, which involved in 465 specimens collected from bats, pangolins, monkeys, and other species. The diverse RNA viruses were analyzed, especially focusing on the genome organization, genetic divergence and phylogenetic relationships  ResultsIn this study, we investigated the viral composition of eight libraries from bats, pangolins, monkeys, and other species, and found several diverse RNA viruses, including the Alphacoronavirus from bat specimens. By characterizing the genome organization, genetic divergence, and phylogenetic relationships, we identified five Alphacoronavirus strains, which shared phylogenetic association with Bat-CoV-HKU8-related strains. The pestivirus-like virus related to recently identified Dongyang pangolin virus (DYPV) strains from dead pangolin specimens, suggesting that these viruses are evolving. Some genomes showed higher divergence from known species (e.g., calicivirus CS9-Cali-YN-CHN-2020), and many showed evidence of recombination events with unknown or known strains (e.g., mamastroviruses BF2-astro-YN-CHN-2020 and EV-A122 AKM5-YN-CHN-2020). The newly identified viruses showed extensive changes and could be assigned as new species, or even genus (e.g., calicivirus CS9-Cali-YN-CHN-2020 and iflavirus Ifla-YN-CHN-2020). Moreover, we identified several highly divergent RNA viruses and estimated their evolutionary characteristics among different hosts, providing data for further examination of their evolutionary dynamics  ConclusionOverall, our study emphasizes the close association between emerging viruses and infectious diseases, and the need for more comprehensive surveys

**DOI:** [10.3389/fmicb.2022.1019444](https://doi.org/10.3389/fmicb.2022.1019444)
**URL:** https://doi.org/10.3389/fmicb.2022.1019444
**Topics:** bats, pangolins, monkeys
</details>


### Fungi <a id='paper-fungi'></a>
<details>
<summary>**The virome from a collection of endomycorrhizal fungi reveals new viral taxa with unprecedented genome organization**</summary>

Mutualistic plant-associated fungi are recognized as important drivers in plant evolution, diversity, and health. The discovery that mycoviruses can take part and play important roles in symbiotic tripartite interactions has prompted us to study the viromes associated with a collection of ericoid and orchid mycorrhizal (ERM and ORM, respectively) fungi. Our study, based on high-throughput sequencing of transcriptomes (RNAseq) from fungal isolates grown in axenic cultures, revealed in both ERM and ORM fungi the presence of new mycoviruses closely related to already classified virus taxa, but also new viruses that expand the boundaries of characterized RNA virus diversity to previously undescribed evolutionary trajectories. In ERM fungi, we provide first evidence of a bipartite virus, distantly related to narnaviruses, that splits the RNA-dependent RNA polymerase (RdRP) palm domain into two distinct proteins, encoded by each of the two segments. Furthermore, in one isolate of the ORM fungus Tulasnella spp. we detected a 12‚Äâkb genomic fragment coding for an RdRP with features of bunyavirus-like RdRPs. However, this 12‚Äâkb genomic RNA has the unique features, for Bunyavirales members, of being tri-cistronic and carrying ORFs for the putative RdRP and putative nucleocapsid in ambisense orientation on the same genomic RNA. Finally, a number of ORM fungal isolates harbored a group of ambisense bicistronic viruses with a genomic size of around 5‚Äâkb, where we could identify a putative RdRP palm domain that has some features of plus strand RNA viruses; these new viruses may represent a new lineage in the Riboviria, as they could not be reliably assigned to any of the branches in the recently derived monophyletic tree that includes most viruses with an RNA genome

**DOI:** [10.1093/ve/veaa076](https://doi.org/10.1093/ve/veaa076)
**URL:** https://doi.org/10.1093/ve/veaa076
**Topics:** fungi
</details>


<details>
<summary>**Appearance of mycovirus-like double-stranded RNAs in the white root rot fungus, Rosellinia necatrix, in an apple orchard**</summary>

Appearance of mycovirus-like double-stranded RNAs in the white root rot fungus,Rosellinia necatrix, in an apple orchard

**DOI:** [10.1111/j.1574-6941.2012.01454.x](https://doi.org/10.1111/j.1574-6941.2012.01454.x)
**URL:** https://doi.org/10.1111/j.1574-6941.2012.01454.x
**Topics:** fungi
</details>


<details>
<summary>**Novel Mycoviruses Discovered in the Mycovirome of a Necrotrophic Fungus**</summary>

The results obtained here have expanded our knowledge of mycoviral diversity, horizontal transfers, and putative cross-kingdom events. To date, this study presents the most extensive and wide diversity collection of mycoviruses infecting the necrotrophic fungusB. cinerea

**DOI:** [10.1128/mbio.03705-20](https://doi.org/10.1128/mbio.03705-20)
**URL:** https://doi.org/10.1128/mbio.03705-20
**Topics:** fungi
</details>


<details>
<summary>**Three new clades of putative viral RNA-dependent RNA polymerases with rare or unique catalytic triads discovered in libraries of ORFans from powdery mildews and the yeast of oenological interest Starmerella bacillaris**</summary>

High throughput sequencing allowed the discovery of many new viruses and viral organizations increasing our comprehension of virus origin and evolution. Most RNA viruses are currently characterized through similarity searches of annotated virus databases. This approach limits the possibility to detect completely new virus-encoded proteins with no detectable similarities to existing ones, i.e. ORFan proteins. A strong indication of the ORFan viral origin in a metatranscriptome is the lack of DNA corresponding to an assembled RNA sequence in the biological sample. Furthermore, sequence homology among ORFans and evidence of co-occurrence of these ORFans in specific host individuals provides further indication of a viral origin. Here, we use this theoretical framework to report the finding of three conserved clades of protein-coding RNA segments without a corresponding DNA in fungi. Protein sequence and structural alignment suggest these proteins are distantly related to viral RNA-dependent RNA polymerases (RdRP). In these new putative viral RdRP clades, no GDD catalytic triad is present, but the most common putative catalytic triad is NDD and a clade with GDQ, a triad previously unreported at that site. SDD, HDD, and ADD are also represented. For most members of these three clades, we were able to associate a second genomic segment, coding for a protein of unknown function. We provisionally named this new group of viruses ormycovirus. Interestingly, all the members of one of these sub-clades (gammaormycovirus) accumulate more minus sense RNA than plus sense RNA during infection

**DOI:** [10.1093/ve/veac038](https://doi.org/10.1093/ve/veac038)
**URL:** https://doi.org/10.1093/ve/veac038
**Topics:** fungi, RdRp
</details>


<details>
<summary>**Plant-feeding insects harbor double-stranded RNA viruses encoding a novel proline-alanine rich protein and a polymerase distantly related to that of fungal viruses**</summary>

Plant-feeding insects harbor double-stranded RNA viruses encoding a novel proline-alanine rich protein and a polymerase distantly related to that of fungal viruses

**DOI:** [10.1016/j.virol.2010.05.015](https://doi.org/10.1016/j.virol.2010.05.015)
**URL:** https://doi.org/10.1016/j.virol.2010.05.015
**Topics:** fungi, Insects, plants
</details>


### Mollosca <a id='paper-mollosca'></a>
<details>
<summary>**A Needle in A Haystack: Tracing Bivalve-Associated Viruses in High-Throughput Transcriptomic Data**</summary>

Bivalve mollusks thrive in environments rich in microorganisms, such as estuarine and coastal waters, and they tend to accumulate various particles, including viruses. However, the current knowledge on mollusk viruses is mainly centered on few pathogenic viruses, whereas a general view of bivalve-associated viromes is lacking. This study was designed to explore the viral abundance and diversity in bivalve mollusks using transcriptomic datasets. From analyzing RNA-seq data of 58 bivalve species, we have reconstructed 26 nearly complete and over 413 partial RNA virus genomes. Although 96.4% of the predicted viral proteins refer to new viruses, some sequences belong to viruses associated with bivalve species or other marine invertebrates. We considered short non-coding RNAs (sncRNA) and post-transcriptional modifications occurring specifically on viral RNAs as tools for virus host-assignment. We could not identify virus-derived small RNAs in sncRNA reads obtained from the oyster sample richest in viral reads. Single Nucleotide Polymorphism (SNP) analysis revealed 938 A-to-G substitutions occurring on the 26 identified RNA viruses, preferentially impacting the AA di-nucleotide motif. Under-representation analysis revealed that the AA motif is under-represented in these bivalve-associated viruses. These findings improve our understanding of bivalve viromes, and set the stage for targeted investigations on the specificity and dynamics of identified viruses

**DOI:** [10.3390/v11030205](https://doi.org/10.3390/v11030205)
**URL:** https://doi.org/10.3390/v11030205
**Topics:** mollosca
</details>


### Hepacivirus <a id='paper-hepacivirus'></a>
<details>
<summary>**Divergent hepaciviruses, delta-like viruses, and a chu-like virus in Australian marsupial carnivores (dasyurids)**</summary>

Although Australian marsupials are characterised by unique biology and geographic isolation, little is known about the viruses present in these iconic wildlife species. The Dasyuromorphia are an order of marsupial carnivores found only in Australia that include both the extinct Tasmanian tiger (Thylacine) and the highly threatened Tasmanian devil. Several other members of the order are similarly under threat of extinction due to habitat loss, hunting, disease, and competition and predation by introduced species such as feral cats. We utilised publicly available RNA-seq data from the NCBI Sequence Read Archive (SRA) database to document the viral diversity within four Dasyuromorph species. Accordingly, we identified 15 novel virus sequences from five DNA virus families (Adenoviridae, Anelloviridae, Gammaherpesvirinae, Papillomaviridae and Polyomaviridae) and three RNA virus taxa: the order Jingchuvirales, the genus Hepacivirus, and the delta-like virus group. Of particular note was the identification of a marsupial specific clade of delta-like viruses that may indicate an association of deltaviruses with marsupial species. In addition, we identified a highly divergent hepacivirus in a numbat liver transcriptome that falls outside of the larger mammalian clade. We also detect what may be the first Jingchuvirales virus in a mammalian host ‚Äì a chu-like virus in Tasmanian devils ‚Äì thereby expanding the host range beyond invertebrates and ectothermic vertebrates. As many of these Dasyuromorphia species are currently being used in translocation efforts to reseed populations across Australia, understanding their virome is of key importance to prevent the spread of viruses to naive populations

**DOI:** [10.1093/ve/vead061](https://doi.org/10.1093/ve/vead061)
**URL:** https://doi.org/10.1093/ve/vead061
**Topics:** hepacivirus, marsupial
</details>


### Serratus <a id='paper-serratus'></a>
<details>
<summary>**Petabase-scale sequence alignment catalyses viral discovery**</summary>

Petabase-scale sequence alignment catalyses viral discovery

**DOI:** [10.1038/s41586-021-04332-2](https://doi.org/10.1038/s41586-021-04332-2)
**URL:** https://serratus.io/
**Topics:** Serratus, Cloud
</details>


### Diptera <a id='paper-diptera'></a>
<details>
<summary>**Novel virus discovery and genome reconstruction from field RNA samples reveals highly divergent viruses in dipteran hosts**</summary>

We investigated whether small RNA (sRNA) sequenced from field-collected mosquitoes and chironomids (Diptera) can be used as a proxy signature of viral prevalence within a range of species and viral groups, using sRNAs sequenced from wild-caught specimens, to inform total RNA deep sequencing of samples of particular interest. Using this strategy, we sequenced from adult Anopheles maculipennis s.l. mosquitoes the apparently nearly complete genome of one previously undescribed virus related to chronic bee paralysis virus, and, from a pool of Ochlerotatus caspius and Oc. detritus mosquitoes, a nearly complete entomobirnavirus genome. We also reconstructed long sequences (1503-6557 nt) related to at least nine other viruses. Crucially, several of the sequences detected were reconstructed from host organisms highly divergent from those in which related viruses have been previously isolated or discovered. It is clear that viral transmission and maintenance cycles in nature are likely to be significantly more complex and taxonomically diverse than previously expected.

**DOI:** [10.1371/journal.pone.0080720](https://doi.org/10.1371/journal.pone.0080720)
**URL:** https://doi.org/10.1371/journal.pone.0080720
**Topics:** diptera
</details>


### Orthomyxoviridae <a id='paper-orthomyxoviridae'></a>
<details>
<summary>**Accumulated metagenomic studies reveal recent migration, whole genome evolution, and undiscovered diversity of orthomyxoviruses**</summary>

Metagenomic studies have uncovered many novel viruses by looking beyond hosts of public health or economic interest. However, the resulting viral genomes are often incomplete, and analyses largely characterize the distribution of viruses over their dynamics. Here, we integrate accumulated data from metagenomic studies to reveal geographic and evolutionary dynamics in a case study ofOrthomyxoviridae, the RNA virus family that includes influenza virus. First, we use sequences of the orthomyxovirid W«îh√†n mosquito virus 6 to track the migrations of its host. We then look at orthomyxovirus genome evolution, finding gene gain and loss across members of the family, especially in the surface proteins responsible for cell and host tropism. We find that the surface protein of W«îh√†n mosquito virus 6 exhibits accelerated non-synonymous evolution suggestive of antigenic evolution, i.e., vertebrate infection, and belongs to a wider quaranjavirid group bearing highly diverged surface proteins. Finally, we quantify the progress of orthomyxovirus discovery and forecast that many divergedOrthomyxoviridaemembers remain to be found. We argue that continued metagenomic studies will be fruitful for understanding the dynamics, evolution, ecology of viruses, and their hosts, regardless of whether novel viruses are identified or not, as long as study designs allowing for the resolution of complete viral genomes are employed The number of known virus species has increased dramatically through metagenomic studies, which search genetic material sampled from a host for non-host genes. Here, we focus on an important viral family that includes influenza viruses, the  Orthomyxoviridae, with over 100 recently discovered viruses infecting hosts from humans to fish. We find that one virus called W«îh√†n mosquito virus 6, discovered in mosquitoes in China, has spread across the globe very recently. Surface proteins used to enter cells show signs of rapid evolution in W«îh√†n mosquito virus 6 and its relatives which suggests an ability to infect vertebrate animals. We compute the rate at which new orthomyxovirus species discovered add evolutionary history to the tree of life, predict that many viruses remain to be discovered, and discuss what appropriately designed future studies can teach us about how diseases cross between continents and species

**DOI:** [10.1128/jvi.01056-23](https://doi.org/10.1128/jvi.01056-23)
**URL:** https://doi.org/10.1128/jvi.01056-23
**Topics:** orthomyxoviridae
</details>


### Resource <a id='paper-resource'></a>
<details>
<summary>**Identification of RNA Virus-Derived RdRp Sequences in Publicly Available Transcriptomic Data Sets**</summary>

RNA viruses are abundant and highly diverse and infect all or most eukaryotic organisms. However, only a tiny fraction of the number and diversity of RNA virus species have been catalogued. To cost-effectively expand the diversity of known RNA virus sequences, we mined publicly available transcriptomic data sets. We developed 77 family-level Hidden Markov Model profiles for the viral RNA-dependent RNA polymerase (RdRp)‚Äîthe only universal ‚Äúhallmark‚Äù gene of RNA viruses. By using these to search the National Center for Biotechnology Information Transcriptome Shotgun Assembly database, we identified 5,867 contigs encoding RNA virus RdRps or fragments thereof and analyzed their diversity, taxonomic classification, phylogeny, and host associations. Our study expands the known diversity of RNA viruses, and the 77 curated RdRp Profile Hidden Markov Models provide a useful resource for the virus discovery community

**DOI:** [10.1093/molbev/msad060](https://doi.org/10.1093/molbev/msad060)
**URL:** https://doi.org/10.1093/molbev/msad060
**Topics:** resource, pHMM, Orthomyxoviridae, splicing
</details>


### Review <a id='paper-review'></a>
<details>
<summary>**Exploring the virome: An integral part of human health and disease**</summary>

Exploring the virome: An integral part of human health and disease

**DOI:** [10.1016/j.prp.2024.155466](https://doi.org/10.1016/j.prp.2024.155466)
**URL:** https://doi.org/10.1016/j.prp.2024.155466
**Topics:** review
</details>


<details>
<summary>**Origins and Evolution of the Global RNA Virome**</summary>

The majority of the diverse viruses infecting eukaryotes have RNA genomes, including numerous human, animal, and plant pathogens. Recent advances of metagenomics have led to the discovery of many new groups of RNA viruses in a wide range of hosts. These findings enable a far more complete reconstruction of the evolution of RNA viruses than was attainable previously. This reconstruction reveals the relationships between different Baltimore classes of viruses and indicates extensive transfer of viruses between distantly related hosts, such as plants and animals. These results call for a major revision of the existing taxonomy of RNA viruses

**DOI:** [10.1128/mBio.02329-18](https://doi.org/10.1128/mBio.02329-18)
**URL:** https://doi.org/10.1128/mBio.02329-18
**Topics:** review, phylogeny
</details>


### Reptiles <a id='paper-reptiles'></a>
<details>
<summary>**The diverse liver viromes of Australian geckos and skinks are dominated by hepaciviruses and picornaviruses and reflect host taxonomy and habitat**</summary>

Lizards have diverse ecologies and evolutionary histories, and represent a promising group to explore how hosts shape virome structure and virus evolution. Yet, little is known about the viromes of these animals. In Australia, squamates (lizards and snakes) comprise the most diverse order of vertebrates, and Australia hosts the highest diversity of lizards globally, with the greatest breadth of habitat use. We used meta-transcriptomic sequencing to determine the virome of nine co-distributed, tropical lizard species from three taxonomic families in Australia and analyzed these data to identify host traits associated with viral abundance and diversity. We show that lizards carry a large diversity of viruses, identifying more than thirty novel, highly divergent vertebrate-associated viruses. These viruses were from nine viral families, including several that contain well known pathogens, such as the Flaviviridae, Picornaviridae, Bornaviridae, Iridoviridae, and Rhabdoviridae. Members of the Flaviviridae were particularly abundant across species sampled here, largely belonging to the genus Hepacivirus: fourteen novel hepaciviruses were identified, broadening the known diversity of this group and better defining its evolution by uncovering new reptilian clades. The evolutionary histories of the viruses studied here frequently aligned with the biogeographic and phylogenetic histories of the hosts, indicating that exogenous viruses may help infer host evolutionary history if sampling is strategic and sampling density high enough. Notably, analysis of alpha and beta diversity revealed that virome composition and richness in the animals sampled here was shaped by host taxonomy and habitat. In sum, we identified a diverse range of reptile viruses that broadly contributes to our understanding of virus-host ecology and evolution

**DOI:** [10.1093/ve/veae044](https://doi.org/10.1093/ve/veae044)
**URL:** https://academic.oup.com/ve/advance-article/doi/10.1093/ve/veae044/7683744?login=false
**Topics:** reptiles, hepacivirus, picornavirus
</details>


### Methodology <a id='paper-methodology'></a>
<details>
<summary>**New lineages of RNA viruses from clinical isolates of Rhizopus microsporus revealed by fragmented and primer-ligated dsRNA sequencing (FLDS) analysis**</summary>

Rhizopus microsporus  is a species in the order Mucorales that is known to cause mucormycosis, but it is poorly understood as a host of viruses. Here, we examined 25 clinical strains of  R. microsporus  for viral infection with a conventional double-stranded RNA (dsRNA) assay using agarose gel electrophoresis (AGE) and the recently established fragmented and primer-ligated dsRNA sequencing (FLDS) protocol. By AGE, five virus-infected strains were detected. Then, full-length genomic sequences of 12 novel RNA viruses were revealed by FLDS, which were related to the families  Mitoviridae, Narnaviridae,  and  Endornaviridae,  ill-defined groups of single-stranded RNA (ssRNA) viruses with similarity to the established families  Virgaviridae  and  Phasmaviridae,  and the proposed family ‚ÄúAmbiguiviridae.‚Äù All the characterized viruses, except a potential phasmavirid with a negative-sense RNA genome, had positive-sense RNA genomes. One virus belonged to a previously established species within the family  Mitoviridae, whereas the other 11 viruses represented new species or even new genera. These results show that the fungal pathogen  R. microsporus  harbors diverse RNA viruses and extend our understanding of the diversity of RNA viruses in the fungal order Mucorales, division Mucoromycota. Identifying RNA viruses from clinical isolates of  R. microsporus  may expand the repertoire of natural therapeutic agents for mucormycosis in the future         The diversity of mycoviruses in fungal hosts in the division Mucoromycota has been underestimated, mainly within the species    Rhizopus microsporus    . Only five positive-sense RNA genomes had previously been discovered in this species. Because current sequencing methods poorly complete the termini of genomes, we used fragmented and primer-ligated double-stranded RNA sequencing to acquire the full-length genomes. Eleven novel mycoviruses were detected in this study, including the first negative-sense RNA genome reported in    R. microsporus    . Our findings extend the understanding of the viral diversity in clinical strains of Mucoromycota, may provide insights into the pathogenesis and ecology of this fungus, and may offer therapeutic options.

**DOI:** [10.1128/msphere.00345-24](https://doi.org/10.1128/msphere.00345-24)
**URL:** https://doi.org/10.1128/msphere.00345-24
**Topics:** methodology, diversity
</details>


### Algae <a id='paper-algae'></a>
<details>
<summary>**Metatranscriptomic Identification of Diverse and Divergent RNA Viruses in Green and Chlorarachniophyte Algae Cultures**</summary>

Our knowledge of the diversity and evolution of the virosphere will likely increase dramatically with the study of microbial eukaryotes, including the microalgae within which few RNA viruses have been documented. By combining total RNA sequencing with sequence and structural-based homology detection, we identified 18 novel RNA viruses in cultured samples from two major groups of microbial algae: the chlorophytes and the chlorarachniophytes. Most of the RNA viruses identified in the green algae class Ulvophyceae were related to the Tombusviridae and Amalgaviridae viral families commonly associated with land plants. This suggests that the evolutionary history of these viruses extends to divergence events between algae and land plants. Seven Ostreobium sp-associated viruses exhibited sequence similarity to the mitoviruses most commonly found in fungi, compatible with horizontal virus transfer between algae and fungi. We also document, for the first time, RNA viruses associated with chlorarachniophytes, including the first negative-sense (bunya-like) RNA virus in microalgae, as well as a distant homolog of the plant virus Virgaviridae, potentially signifying viral inheritance from the secondary chloroplast endosymbiosis that marked the origin of the chlorarachniophytes. More broadly, these data suggest that the scarcity of RNA viruses in algae results from limited investigation rather than their absence

**DOI:** [10.3390/v12101180](https://doi.org/10.3390/v12101180)
**URL:** https://doi.org/10.3390/v12101180
**Topics:** algae
</details>


<details>
<summary>**Revealing RNA virus diversity and evolution in unicellular algae transcriptomes**</summary>

Remarkably little is known about the diversity and evolution of RNA viruses in unicellular eukaryotes. We screened a total of 570 transcriptomes from the Marine Microbial Eukaryote Transcriptome Sequencing Project that encompasses a wide diversity of microbial eukaryotes, including most major photosynthetic lineages (i.e. the microalgae). From this, we identified thirty new and divergent RNA virus species, occupying a range of phylogenetic positions within the overall diversity of RNA viruses. Approximately one-third of the newly described viruses comprised single-stranded positive-sense RNA viruses from the order Lenarviricota associated with fungi, plants, and protists, while another third were related to the order Ghabrivirales, including members of the protist and fungi-associated Totiviridae. Other viral species showed sequence similarity to positive-sense RNA viruses from the algae-associated Marnaviridae, the double-stranded RNA (ds-RNA) Partitiviridae, as well as tentative evidence for one negative-sense RNA virus related to the Qinviridae. Importantly, we were able to identify divergent RNA viruses from distant host taxa, revealing the ancestry of these viral families and greatly extending our knowledge of the RNA viromes of microalgal cultures. Both the limited number of viruses detected per sample and the low sequence identity to known RNA viruses imply that additional microalgal viruses exist that could not be detected at the current sequencing depth or were too divergent to be identified using sequence similarity. Together, these results highlight the need for further investigation of algal-associated RNA viruses as well as the development of new tools to identify RNA viruses that exhibit very high levels of sequence divergence

**DOI:** [10.1093/ve/veab070](https://doi.org/10.1093/ve/veab070)
**URL:** https://doi.org/10.1093/ve/veab070
**Topics:** algae
</details>


### Vertebrate <a id='paper-vertebrate'></a>
<details>
<summary>**The diversity, evolution and origins of vertebrate RNA viruses**</summary>

The diversity, evolution and origins of vertebrate RNA viruses

**DOI:** [10.1016/j.coviro.2018.07.017](https://doi.org/10.1016/j.coviro.2018.07.017)
**URL:** https://doi.org/10.1016/j.coviro.2018.07.017
**Topics:** vertebrate, phylogeny
</details>


### Flaviviridae <a id='paper-flaviviridae'></a>
<details>
<summary>**Transcriptome mining extends the host range of the Flaviviridae to non-bilaterians**</summary>

The flavivirids (family Flaviviridae) are a group of positive-sense RNA viruses that include well-documented agents of human disease. Despite their importance and ubiquity, the timescale of flavivirid evolution is uncertain. An ancient origin, spanning millions of years, is supported by their presence in both vertebrates and invertebrates and by the identification of a flavivirus-derived endogenous viral element in the peach blossom jellyfish genome (Craspedacusta sowerbii, phylum Cnidaria), implying that the flaviviruses arose early in the evolution of the Metazoa. To date, however, no exogenous flavivirid sequences have been identified in these hosts. To help resolve the antiquity of the Flaviviridae, we mined publicly available transcriptome data across the Metazoa. From this, we expanded the diversity within the family through the identification of 32 novel viral sequences and extended the host range of the pestiviruses to include amphibians, reptiles, and ray-finned fish. Through co-phylogenetic analysis we found cross-species transmission to be the predominate macroevolutionary event across the non-vectored flavivirid genera (median, 68‚Äâper cent), including a cross-species transmission event between bats and rodents, although long-term virus‚Äìhost co-divergence was still a regular occurrence (median, 23‚Äâper cent). Notably, we discovered flavivirus-like sequences in basal metazoan species, including the first associated with Cnidaria. This sequence formed a basal lineage to the genus Flavivirus and was closer to arthropod and crustacean flaviviruses than those in the tamanavirus group, which includes a variety of invertebrate and vertebrate viruses. Combined, these data attest to an ancient origin of the flaviviruses, likely close to the emergence of the metazoans 750‚Äì800 million years ago

**DOI:** [10.1093/ve/veac124](https://doi.org/10.1093/ve/veac124)
**URL:** https://doi.org/10.1093/ve/veac124
**Topics:** Flaviviridae, host range
</details>





## Contributing

We welcome contributions! See our [Contributing Guidelines](CONTRIBUTING.md) for details on how to add new tools or improve existing entries.

### How to Add a Tool

1. Create a new JSON file in the `entries/` directory
2. Follow the schema defined in `scripts/schema.json`
3. Submit a pull request

## License

This project is licensed under the MIT License - see the [LICENSE](LICENSE) file for details.

## Acknowledgments

- Inspired by the [awesome-virome](https://github.com/shandley/awesome-virome) project
- Built with ‚ù§Ô∏è by the RNA virus research community

---

*Last updated: 2025-08-15 16:13:41 UTC*
    